<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>
<meta name="generator" content="HTML Tidy for Linux (vers 25 March 2009), see www.w3.org">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="Content-Type" content="text/html; charset=iso-8859-1">
<meta name="description" content="Interactive Introductory / Foundational Evolution and Molecular Biology">
<meta name="keywords" content="free on-line biology text, interactive molecular biology, interactive text, biology on-line">
<meta name="keywords" content="Biofundamentals, College-level, Foundational concepts, Klymkowsky, Evolution, Molecular Biology, Cell Biology, Interactive">
<meta name="GENERATOR" content="Mozilla/4.75 (Macintosh; U; PPC) [Netscape]">
<title>Biofundamentals @ UC Boulder - Peptide bonds and polypeptides</title>

<script language="JavaScript" type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
<style type="text/css">
<!--
.style1 {color: #000000}
-->
</style>

<script src="../../Scripts/AC_RunActiveContent.js" type="text/javascript">
</script>
<style type="text/css">
<!--
a:link {text-decoration: none}
a:visited {text-decoration: none}
a:active {text-decoration: none}
a:hover {text-decoration: underline}
body,td,th {
        font-family: Arial, Helvetica, sans-serif;
}
.style2 {
        color: #990000;
        font-weight: bold;
}
.style4 {font-size: x-small}
.white {
        color: #FFF;
}
.cayenne {
        color: #800000;
}
-->
</style>

<style type="text/css">
 body {
  background-color: #FCFDFF;
  color: #000000;
 }
 :link { color: #0000FF }
 :visited { color: #0066FF }
 :active { color: #0066FF }
 div.c20 {font-family: Arial, Helvetica, sans-serif; font-size: 64%; text-align: right}
 span.c19 {font-family: Arial, Helvetica, sans-serif; font-size: 64%}
 hr.c18 {text-align: center}
 p.c17 {font-family: Arial, Helvetica, sans-serif; font-size: 64%; text-align: right}
 p.c16 {font-family: Arial, Helvetica, sans-serif; text-align: left}
 table.c15 {background-color: #CCCCCC}
 td.c14 {background-color: #0000FF}
 td.c13 {background-color: #FFFFFF}
 td.c12 {background-color: #990000}
 div.c11 {text-align: center}
 p.c10 {text-align: left}
 span.c9 {color: #990000}
 span.c8 {color: #000000}
 span.c7 {font-family: Arial, Helvetica, sans-serif}
 p.c6 {font-family: Arial, Helvetica, sans-serif}
 div.c5 {text-align: right}
 span.c4 {color: #006600; font-family: Arial, Helvetica, sans-serif}
 div.c3 {text-align: left}
 span.c2 {font-family: Arial, Helvetica, sans-serif; font-size: 80%}
 span.c1 {color: #000000; font-family: Arial, Helvetica, sans-serif}
</style>
</head>
<body onload="MM_preloadImages('../../Biofundamentals-2012/lectureNotes-Revision/AllGraphics/DLMovie.gif','../../Biofundamentals-2012/lectureNotes-Revision/AllGraphics/PrPc acidic.jpg','../../BioFun-Support/AllGraphics/PrPc acidic.jpg')">
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="512">
<div class="c3"><span class="c1"><strong>22. Peptide bonds, polypeptides and proteins</strong></span> <span class="c2"><br></span></div>
</td>
<td width="47"><span class="c4"><strong><a name="top" id="top"></a></strong></span></td>
<td width="112"><a href="https://twitter.com/share" class="twitter-share-button">Tweet</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script> </td>
<td width="24">&nbsp;</td>
<td width="25">
<div class="c5"><a href="Topic3-6_Making Proteins.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowRIGHT.gif" alt="arrow" width="24" height="24" border="0"></a></div>
</td>
</tr>
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c6">We have already mentioned proteins a number of times, since there are few biological processes that do not rely on them.</p>
<p class="c6">The first scientific characterization of what are now known as proteins was published in 1838, the Dutch chemist, Gerardus Johannes Mulder (1802–1880).  </p>
</td>
<td> </td>
<td><img src="../../BioFun-Support/wordles/page18.jpg" width="330" height="140" alt="wordle-1"></td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c6">After an analysis of a number of different substances, he proposed that they all represented versions of a common chemical core, with the molecular formula: C<sub>400</sub>H<sub>620</sub>N<sub>100</sub>O<sub>120</sub>P<sub>1</sub>S<sub>1</sub>. <br>
  <br>
  The differences between proteins were primarily in the numbers of phosphate (P) and sulfur (S) atoms they contained.  The name “protein”, from the Greek word πρώτα (“prota”), meaning “primary” was suggested by the Swede, Jons Jakob Ber- zelius (1779–1848) based on the presumed importance of these compounds in biological systems.  Proteins could be dissolved in either water or dilute salt solutions but aggregated and became insoluble when the solution was heated; as we will see this aggregation reflects a change in protein structure. </p>
<p class="c6">Proteins can act as catalysts (enzymes) and regulators of chemical reactions - we have already seen how proteins act to regulate transport across membranes. Proteins control the expression of genes, how genes respond to internal and external signals, and the replication of the genetic material.</p>
<p><span class="c7">Proteins</span> <span class="c7">can act as structural components, determining both the shape and mechanical properties of cells and tissues.  They can be motors, responsible for movements <strong>within</strong> cells and the movement of cells, tissues and the organism as a whole. </span></p>
<p><span class="c6">Mulder was able to break down a number of proteins through an acid hydrolysis reaction into amino acids, named because they contained amino (-NH2) and carboxylic acid (COOH) groups.  Twenty different amino acids could be identified in hydrolyzed samples of proteins. </span></p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="182">
<p><img src="../../BioFun-Support/AllGraphics/AminoAcid.jpg" alt="" width="182" height="120"></p>
</td>
<td width="10">
<p> </p>
</td>
<td width="508">
<p class="c6">These subunits out to be  α-amino acids that are linked together by peptide bonds into polypeptide chains.</p>
<p>We can think of an amino acid as a modified form of methane (CH<span class="c7"><strong><sub>4</sub></strong></span>).  Instead of four hydrogens attached to the central C, there is one H, an <span class="c7">amino <span class="c8">group</span> (<strong>-NH<sub>2</sub></strong>), a carboxylic acid group (<strong>-COOH</strong>), and a final, variable (R) group attached to the central "α-carbon".</span></p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="503">
<p class="c6">The various R-groups are often referred to as "side-chains".</p>
<p><span class="c7">The four groups attached to the</span> <span class="c7"><span class="style1">α</span>-carbon are arranged at the vertices of a tetrahedron.</span></p>
<p class="c6">If all four groups attached to the <span class="style1">α</span>-carbon are different from one another, as they are in all amino acids except glycine, the amino acid can exist in two possible stereoisomers, which are known as enantiomers.</p>
</td>
<td width="13"> </td>
<td width="184"><a href="http://mathworld.wolfram.com/Tetrahedron.html" target="_blank"><img src="../../BioFun-Support/AllGraphics/tetrahedron.jpg" alt="" width="180" height="180"></a></td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="248"><a href="http://biomedia.bio.purdue.edu/IML/Amino/html/stereoisomers.html" target="_blank" onmouseover="MM_swapImage('Image12','','../../BioFun-Support/AllGraphics/DLMovie.gif',1)" onmouseout="MM_swapImgRestore()"><img src="../../BioFun-Support/AllGraphics/DLAminoAcids.jpg" name="Image12" width="248" height="128" border="0" alt="** PLEASE DESCRIBE THIS IMAGE **" id="Image12"></a></td>
<td width="10"> </td>
<td width="442">
<p class="c6">Enantiomers are mirror images of one another and are termed the L- and D- forms.</p>
<p class="c6">Only L-type amino acids are found in proteins, even though there is <a href="http://www.bio.puc.cl/pap03.htm" target="_blank">no obvious reason</a> that proteins could not have also been made using both types of amino acids, or using D-amino acids.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p><span class="c7">It is not that D-amino acids do not occur in nature, or in organisms, they do. </span><span class="c7">They are found in biomolecules, such as the antibiotic gramicidin, which is composed of alternating L-and D-type amino acids - however gramicidin is synthesized by <a href="http://csbcc1.csbcc.dartmouth.edu/nrps.html" target="_blank">a different process</a> than that used to synthesize proteins.</span></p>
<p><span class="c7">It appears that the universal use of L-type amino acids is yet another example of the evolutionary relatedness of organisms (it is a homologous trait).</span> <span class="c7">Even though there are hundreds of different amino acids known, only 19 amino acids and one imino acid, proline, are found in proteins.</span></p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="280"><iframe width="400" height="300" src="http://www.youtube.com/embed/L9L6clXbJD8?rel=0" frameborder="0"></iframe><br></td>
<td width="10"> </td>
<td width="430">
<p class="c6">These amino acids differ in their R-groups.</p>
<p class="c6">Some of these R-groups are highly hydrophobic, some are hydrophilic, and some of the hydrophilic R-groups are positively or negatively charged at physiological pH.</p>
<p class="c6">The different R-groups provide proteins with a range of chemical properties.</p>
<p>[<a href="../../BioFun-Support/AllGraphics/AminoAcidViewer.swf" target="_blank">link to amino acid viewer</a>]</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c6"><strong>Building a polypeptide: </strong></p>
<p class="c6">As we noted for nucleic acids, a polymer is a chain of subunits, monomers, linked together by chemical bonds. The bond between two amino acids is known as a peptide bond<span class="c8">. Two amino acids, joined together by a peptide bond, is known as a dipeptide<span class="c9"><strong>.</strong></span></span></p>
<p class="c6"><span class="c8">An amino acid polymer is known as a polypeptide</span>. A polypeptide can be composed of 20 different possible monomers that can, in theory, be linked together in any imaginable order.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="490">
<p><span class="c7">The formation of a peptide bond involves what is known as a condensation or sometimes a "dehydration" reaction.  In a condensation reaction</span> two molecules are joined together and a molecule of water is released.</p>
<p><span class="c7">In a hydrolysis reaction</span>, the reverse occurs. <span class="c7">The addition of a water molecule is associated with splitting the original molecule into two parts.</span></p>
</td>
<td width="104"><img src="../../BioFun-Support/AllGraphics/peptideBond.gif" width="348" height="213" alt="** PLEASE DESCRIBE THIS IMAGE **"></td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p>In the case of both nucleic acids and polypeptides, polymer assembly involves a condensation reaction (with the release of water), while polymer disassembly into monomers involves a hydrolysis reaction (with water as a reactant).</p>
<p><span class="c7">Combining amino acids into a polypeptide</span> <span class="c7">can produce a very large number of possible polypeptides. There are 20<sup>100</sup> possible different polypeptides that are 100 amino acid residues long. </span>Many biological polypeptides are much longer!</p>
<p><span class="c7">During the synthesis of a polypeptide, an amino acid is attached to the C- or carboxyl terminus of the existing chain.</span> <span class="c7">This generates an unbranched, linear polymer.</span></p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="480"><iframe width="480" height="360" src="http://www.youtube.com/embed/_5kbJlqInFs?rel=0" frameborder="0"></iframe></td>
<td width="18"> </td>
<td width="222">
<p class="c6">The peptide bond has a number of characteristics that are critical in determining how polypeptides behave.</p>
<p class="c6">Although drawn as a single bond, the peptide bond behaves more like a double bond.</p>
<p class="c6">While there is free rotation around a single bond, rotation around a peptide bond is constrained.</p>
<p class="c10"><span class="c7">Proline peptide bonds are found in the cis</span> configuration ~100 times as often as those between other amino acids.</p>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="140">
<p><img src="../../BioFun-Support/AllGraphics/prolineKink.jpg" alt="" width="140" height="130"></p>
</td>
<td width="10"> </td>
<td width="570">
<p><span class="c7">The carbonyl oxygen</span> (-C=O) acts as a H-bond acceptor, while the amino hydrogen (-N-H) acts as a H-bond donor.</p>
<p><span class="c7">There is <strong>no H-bond donor</strong> in the proline peptide bond</span> <span class="c7">and the presence of proline leads to a bend or kink in the polypeptide chain.</span></p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="485"><iframe width="480" height="360" src="http://www.youtube.com/embed/_xF96sNWnK4?rel=0" frameborder="0"></iframe></td>
<td width="13"> </td>
<td width="222">
<p>In this molecular level simulation, you cannot see the water, but you can watch this single polypeptide chain fold; the spinning of side chains around single bonds is very clear.</p>
<p>The colors indicate where an animo acid is along the polypeptide chain.  </p>
<p>Notice that molecular movement continue even after the molecule reaches its compact (presumably native) state.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table class="c15" width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td class="c12" width="80">
<div class="white c11">
<p>Questions to answer</p>
<p><br></p>
</div>
</td>
<td class="c13" width="640"><span class="c7"> </span>
<ol>
<li>What limits the structure of an R-group? </li>
<li>Are there any theoretical constraints on the order of monomers in a polypeptide?</li>
<li><span class="style5">Consider a condensation reaction; how would the reaction be effected if we removed water from the system?  </span></li>
<li>Why does it matter that rotation around a peptide bond is "constrained"?</li>
</ol>
</td>
</tr>
<tr>
<td class="c14">
<div class="c11"><span class="whoe1 white">Questions to ponder</span><br></div>
</td>
<td class="c13">
<ul>
<li class="style21">Is the use of a common set of amino acids among living organisms an analogous or a homologous trait?</li>
<li class="style21">What factors can you imagine influenced the "universal" set of amino acids used in organisms?</li>
</ul>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c6"><strong>Water, polypeptide synthesis and folding: </strong></p>
<p class="c6">Were it not for the presence of hydrophobic R-groups, all polypeptides would assume an extended configuration in water.  H-bond donors and acceptor groups within the polypeptide backbone would form H-bonds with each other and, because there are so many of them, with water molecules.</p>
<p><span class="c7">Typical polypeptides have all of the different R-groups in various proportions. We write the amino acid sequence of a polypeptide from its N- or amino terminus</span> to its C- or carboxyl terminus, with the N-terminus to the left and the C-terminus to the right. <span class="c7">The sequence of amino acid residues, that is what is left of the amino acid once it is part of the polypeptide, along the length of a polypeptide from its N- to its C-terminus, is known as its primary structure</span>.</p>
<p class="c6">A number of the amino acid R-groups are <a href="http://www.russell.embl-heidelberg.de/aas/hydrophobic.html" target="_blank">hydrophobic</a>. Their presence makes an extended configuration for the polypeptide energetically unfavorable.</p>
<p><span class="c7">Very much like the process that drives the assembly of lipids into micelles and bilayers, a typical polypeptide in aqueous solution will collapse onto itself in order to remove as many of its hydrophobic residues</span> from contact with water as possible (otherwise, water molecules would be organized around the hydrophobic residues, leading to a decrease in entropy!)<br>
<br></p>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="513">
<p><span class="c7">It is a basic assumption of structural biology that the final folded state of a polypeptide is determined by the sequence of amino acids along its length, and that this final structure is the state of lowest, or close to the lowest, energ</span>y.</p>
<p><span class="c7">Protein structure is commonly presented in a hierarchical</span> manner. <span class="c7">While this is an over-simplification, it is a good place to start.</span></p>
</td>
<td class="c13" width="10"> </td>
<td width="200"><iframe width="400" height="300" src="http://www.youtube.com/embed/Rq7DwrX0Uoc?rel=0" frameborder="0"></iframe></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c10"> </p>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c10"><span class="c7">As it is being synthesized, the process known as translation</span>, the polypeptide chain begins to fold. <span class="c7">We can think of the folding process as a walk across an energy landscape. </span></p>
<p class="c10"><span class="c7">The goal is to find the lowest point in the landscape, the energy minimum of the system.</span> <span class="c7">This is generally assumed to be the native</span> or functional state of the polypeptide.</p>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c10"> </p>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="240">
<p><img src="../../BioFun-Support/AllGraphics/FoldingLandscape.jpg" width="240" height="227" alt="** PLEASE DESCRIBE THIS IMAGE **"></p>
</td>
<td width="15"> </td>
<td width="445">
<div class="c5">
<p class="c10"><span class="c7">The first step is the movement of hydrophobic R-groups out of contact with water. </span><span class="c7">This drives the collapse of the polypeptide into a compact and dynamic "molten globule</span>".</p>
<p class="c16">The path to the native state is not necessarily a smooth or predetermined one.</p>
<p class="c16">The folding polypeptide can get "stuck" in a local energy minimum; there may not be enough energy (derived from thermal collisions) for it to get out again. </p>
</div>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="469">
<p><span class="c7">If a polypeptide gets stuck, there are mechanisms to unfold it and let it try again to reach its native state. This process of partial unfolding is carried out by proteins known as chaperones</span>.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c10"> </p>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="281">
<p class="c10"><a href="http://www.pnas.org/cgi/content/abstract/98/6/2985" target="_blank" onmouseover="MM_swapImage('Image13','','../../BioFun-Support/AllGraphics/PrPc acidic.jpg',1)" onmouseout="MM_swapImgRestore()"><img src="../../BioFun-Support/AllGraphics/PrPc%20neutral.jpg" alt="protein" name="Image13" width="281" height="287" border="0" id="Image13"></a></p>
</td>
<td width="10"> </td>
<td width="429">
<p class="c16">Some amino acid R-groups contain carboxylic acid or amino groups, and so also act as weak acids and bases.</p>
<p class="c16">Depending on the pH of the solution they are in, these groups may be protonated or unprotonated (that is, charged).</p>
<p class="c16">Whether a group is charged or uncharged can have a dramatic effect on the structure, and therefore the activity, of a protein.</p>
<p class="c16">By regulating pH, an organism can modulate the activity of specific proteins. There are, in fact, compartments within eukaryotic cells that are maintained at low pH in part to regulate protein structure and protein activity.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c6"><strong>Secondary structure features: </strong></p>
<p class="c6">All polypeptides share a common backbone structure made up of peptide bonds. It is therefore not surprising that there are common patterns in polypeptide folding.</p>
<p class="c6">The first of these to be recognized, the α-heIix, was discovered by <a href="http://www.paulingexhibit.org/exhibit/alpha-helix.html" target="_blank">Linus Pauling</a> and Robert Corey, and reported in 1951.</p>
<p class="c16">This was followed shortly thereafter by their description of the β-sheet.</p>
<p class="c16">The forces that drive the formation of the α-helix and the β-sheet will be familiar. They are they same forces that underlie water structure.</p>
<p class="c17">Linus at age 5</p>
</td>
<td> </td>
<td><a href="http://www.orst.edu/dept/lpi/lpbio/lpbio2.html" target="_blank"><img src="../../BioFun-Support/AllGraphics/youngLinus.jpg" width="103" height="280" border="0" alt="** PLEASE DESCRIBE THIS IMAGE **"></a></td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="119">
<p><img src="../../BioFun-Support/AllGraphics/alphaHelix.jpg" width="113" height="200" alt="** PLEASE DESCRIBE THIS IMAGE **"></p>
</td>
<td width="11"> </td>
<td width="389">
<p><span class="c7">In an</span> <span class="c7">α-helix and a β-sheet, all of the possible H-bonds involving the peptide bond's donor and acceptor groups are formed within the polypeptide.</span></p>
<p class="c6">This stabilizes the structure.</p>
<p class="c6">In an α-helix , the R-groups point outward from the long axis of the helix.</p>
</td>
<td width="181">
<div class="c5"><img src="../../BioFun-Support/AllGraphics/axialhelix.jpg" width="180" height="191" alt="** PLEASE DESCRIBE THIS IMAGE **"></div>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
<tr>
<td> </td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p><span class="c7">In β-sheets, the H-bonds between the peptide bond donor and acceptor groups are made between either parallel</span> or anti-parallel polypeptide chains.</p>
<p class="c17">These are anti-parallel ß-sheets</p>
<p class="c16">The R-groups point out of and into the plane of the sheet.</p>
</td>
<td> </td>
<td><img src="../../BioFun-Support/AllGraphics/betasheet.gif" width="429" height="256" alt="** PLEASE DESCRIBE THIS IMAGE **"></td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c6">A key factor that determines the specific details of polypeptide folding are the interactions between R-groups, not only with one another but with water and dissolved ions.</p>
<p class="c6">In a water-soluble polypeptide, most but not necessarily all of the surface R-groups will be hydrophilic. Hydrophobic groups will tend to be buried in the polypeptide's interior. </p>
<p><span class="c7">Some polypeptides are inserted into membranes.</span> <span class="c7">In these polypeptides, hydrophobic R-groups on the surface of the folded polypeptide will interact with the hydrophobic interior of the lipid bilayer.</span> (Does that make sense to you, thermodynamically?)</p>
<hr class="c18" width="60%" size="1" noshade>
<p class="c6"><strong>Other factors in protein structure: </strong> In addition to hydrophobic/hydrophilic effects, ionic interactions between acidic and basic R-groups, van der Waals interactions and H-bonds also play an important role in determining the tertiary or three dimensional structure of the polypeptide.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="94">
<p><img src="../../BioFun-Support/AllGraphics/vitamin-B12.jpg" width="96" height="120" alt="b12"></p>
</td>
<td width="13"> </td>
<td width="613">
<p class="c6">Proteins can include non-amino acid-based components, known generically as co-factors.  A protein minus its cofactors is known as an apoprotein.  Together with its cofactors, it is known as a holoprotein. Generally, without its cofactors, a protein is inactive (and often unstable).</p>
<p><span class="c7">Cofactors can range in complexity from a single metal atom to quite complex molecules, such as <a href="http://upload.wikimedia.org/wikipedia/commons/thumb/e/ec/Cobalamin.png/280px-Cobalamin.png" target="_blank">← vitamin B<sub>12</sub></a></span>.  The retinal group of bacteriorhodopsin and the heme group (with its central iron ion, are also a co-factors. </p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c6">In general, a gene does not necessarily encode a protein, but rather a polypeptide.  A protein is a functional entity. In many cases a protein is composed of multiple polypeptides.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="571" height="13">
<p>For example, the membrane bound ATP synthase → we talked about previously is composed of  8 different types of polypeptides, many of which are present in multiple copies assembled to form the final, functional protein. </p>
<p><span class="c7">A protein that contains multiple polypeptides, whether of the same or different types, is said to have a quaternary</span> structure.</p>
<p class="c16">There are also higher levels of polypeptide organization. For example, a particular polypeptide may be a component of a number of different proteins.</p>
</td>
<td width="10"> </td>
<td width="119"><a href="http://www.atpsynthase.info/FAQ.html" target="_blank"><img src="../../BioFun-Support/AllGraphics/ATPsynthase.jpg" width="120" height="160" alt="atp synthase"></a></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1"></td>
</tr>
</table>
<table class="c15" width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td class="c12" width="80">
<div class="c11">
<p class="white">Questions to answer</p>
<p class="white"> </p>
<p class="white"><br></p>
</div>
</td>
<td class="c13" width="640"><span class="c7"> </span>
<ol>
<li><span class="style5">How might changing the pH of the solution alter a protein's structure and activity?</span></li>
<li>What happens to a typical protein if you place it in a hydrophobic solvent?</li>
<li>What would be the structure of a polypeptide if all of its R-groups were hydrophilic? </li>
<li>How might a chaperone recognize a misfolded polypeptide? </li>
<li>What might the role of chaperones be in the assembly of a protein composed of multiple polypeptides?</li>
<li>Summarize the differences in structure between a protein that is soluble in the cytoplasm and one that is buried in the membrane.</li>
<li>Why might proteins that require co-factors misfold in the absence of the co-factor?</li>
<li>How could surface hydrophobic R-groups facilitate protein-protein interactions.</li>
</ol>
</td>
</tr>
<tr>
<td class="c14">
<div class="c11"><span class="whoe1 white">Questions to ponder</span><br></div>
</td>
<td class="c13">
<ul>
<li class="style21">How does increasing entropy drive protein assembly?</li>
<li class="style21">Suggest some reasons why cofactors are necessary in biological systems (proteins)?</li>
</ul>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1"></td>
</tr>
</table>
<table width="760" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td>replace with revised beSocratic activity</td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
<tr>
<td>
<hr></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="21"><a href="#TOP"><span class="c19"><img src="../../BioFun-Support/AllGraphics/ArrowUP.GIF" width="24" height="24" alt="** PLEASE DESCRIBE THIS IMAGE **"></span></a><a href="Topic3-4_Mutation.htm" target="_self"></a></td>
<td width="55">
<div class="c3"><a href="#TOP"></a></div>
</td>
<td width="615">
<div class="c20">modified <!-- #BeginDate format:En2 -->21-Jun-2014<!-- #EndDate --></div>
</td>
<td width="29">
<div class="c5"><a href="Topic3-6_Making Proteins.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowRIGHT.gif" width="24" height="24" border="0" alt="** PLEASE DESCRIBE THIS IMAGE **"></a></div>
</td>
</tr>
</table>
<script type='text/javascript'>
var _hl=_hl||{};_hl.site='2622';(function(){var hl=document.createElement('script');hl.type='text/javascript';hl.async=true;hl.src=document.location.protocol+'//highlighter.com/webscript/v1/js/highlighter.js';var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(hl,s);})();
</script>
</body>
</html>