
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" 
	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
<meta name="Content-Type" content="text/html; charset=iso-8859-1" />
<meta name="Description" content="Interactive Introductory / Foundational Evolution and Molecular Biology" />
<meta name="Keywords" content="free on-line biology text, interactive molecular biology, interactive text, biology on-line" /><meta name="" content="free on-line biology, molecular biology, text" />
<meta name="Keywords" content="Biofundamentals, College-level, Foundational concepts, Klymkowsky, Evolution, Molecular Biology, Cell Biology, Interactive" />
   <meta name="GENERATOR" content="Mozilla/4.75 (Macintosh; U; PPC) [Netscape]" />

<title>BioFundamentals @ UC Boulder - Selection and drift</title>
<script language="JavaScript" type="text/javascript">
<!--



function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
//-->
</script>
</head>
<head>
<style type="text/css">
<!--
a:link {text-decoration: none}
a:visited {text-decoration: none}
a:active {text-decoration: none}
a:hover {text-decoration: underline}

.white {
	color: #FFF;
}
</style>
</head>
<body bgcolor="#FCFDFF" text="#000000" link="#0000FF" vlink="#0066FF" alink="#0066FF">
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="572"><font face="Arial, Helvetica, sans-serif"><b><strong>7. Non-adaptive
                    evolutionary events: sampling effects</strong><font color="#006600" face="Arial, Helvetica, sans-serif"><a name="top" id="top2"></a></font><font face="Arial, Helvetica, sans-serif" size="-1"><br />
    </font></b></font></td>
    <td width="84"><a href="https://twitter.com/share" class="twitter-share-button">Tweet</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script></td>
    <td width="19"><font face="Arial, Helvetica, sans-serif"><b><a href="Topic1G_Evo.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="arrow" width="24" height="24" border="0" /></a></b></font></td>
    <td width="25"><div align="right"><a href="Topic1I_Evo.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowRIGHT.gif" alt="arrow" width="24" height="24" border="0" /></a></div></td>
    <table width="700" border="0" cellspacing="0" cellpadding="0">
    <tr>
        <td><hr size="1" noshade="noshade" /></td>
    </tr>
</table>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td width="301"><div align="left"><img src="../../BioFun-Support/wordles/page8.jpg" width="300" height="90" alt="wordle-1" /></div></td>
    <td width="10">&nbsp;</td>
    <td width="409"><p><font face="Arial, Helvetica, sans-serif">You might think
        that all evolutionary  processes are by their very nature adaptive,
      but this is not the case. </font></p>
        <p><font face="Arial, Helvetica, sans-serif">Accidents
            can &quot;select&quot; a small subset of organisms from the larger
        population.</font></p></td>
  </tr>
</table>

<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr> 
    <td> <p><font face="Arial, Helvetica, sans-serif">Founder
        effects and evolutionary bottlenecks occur when a new population
is based on a small, randomly selected group of individuals. </font></p>
        <p><font face="Arial, Helvetica, sans-serif">The founder
            effect applies when a small group of
            individuals first colonizes a new and isolated territory,
            such as an island. An evolutionary
            bottleneck occurs
            when some disaster or disease quickly reduces a once large population
        to a much smaller one. </font><font face="Arial, Helvetica, sans-serif">The
        original, large population is likely to have had a large and diverse
        set of alleles within it. </font><font face="Arial, Helvetica, sans-serif">The
        smaller founder/post-bottleneck populations will often have, by chance,
        a less diverse set of alleles.&nbsp;</font></p></td>
    </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
    <tr>
        <td><p align="left" class="style1"><font face="Arial, Helvetica, sans-serif">it
                    is possible that alleles that were rare in the original population
                    will be more common in the founder/post-bottleneck population.&nbsp; </font></p>
            <p align="left" class="style1"><font face="Arial, Helvetica, sans-serif">These
                    traits may or may not be adaptive in the new environment;
                    in some cases they can be detrimental, and mutation and natural
                    selection will then work to eliminate them or ameliorate
            their effects.</font></p></td>
        <td>&nbsp;</td>
        <td><img src="../../BioFun-Support/AllGraphics/bottleneck.jpg" alt="bottleneck" width="328" height="189" /></td>
    </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
    <tr>
        <td><p align="left"><font face="Arial, Helvetica, sans-serif">There
                    is strong evidence for a severe evolutionary bottleneck during
                    the course of <a href="http://www.dubage.com/API/ThePolymath/1.1/ThePolymath0701ebam.html" target="_blank">human
        evolution</a>.&nbsp;&nbsp; Once a
                    population is small, the effects of <span class="style7">genetic
                    drift</span> will also increase and can become  profound. </font></p>
            <p align="left"><em>&quot;If a population is finite in size (as all populations
                are) and if a given pair of parents have only a small number
                of offspring, then <strong>even in the absence of all selective
                forces</strong>, the frequency of a gene will not be exactly
                reproduced in the next generation because of sampling error.
              &nbsp; </em></p>
            <p><em>If in a population of 1000 individuals the frequency of &quot;a&quot; is
                0.5 in one generation, then it may by chance be 0.493 or 0.505
                in the next generation because of the chance production of a
                few more or less progeny of each genotype. </em></p>
            <p><em>In the second generation, there is another sampling error
                    based on the new gene frequency, so the frequency of &quot;a&quot; may
                go from 0.505 to 0.501 or back to 0.498. This process of random
                fluctuation continues generation after generation, with no force
                pushing the frequency back to its initial state because the population
                has no &quot;genetic memory&quot; of its state many generations
                ago. </em></p>
            <p><em>Each generation is an independent event. The final result of this
                random change in<strong> allele frequency is that the population
                eventually drifts to p=1 or p=0</strong>.&nbsp; After this point,
                no further change is possible; the population has become homozygous.&nbsp;</em></p>
            <p><em> A different population, isolated from the first, also undergoes
                this random genetic drift, but it may become homozygous for allele &quot;A&quot;,
                whereas the first population has become homozygous for allele &quot;a&quot;.
                &nbsp; </em></p>
            <p><em>As time goes on, isolated populations diverge from each other,
                each losing heterozygosity. The variation originally present
    <strong>within</strong> populations now appears as variation <strong>between</strong> populations.&quot; </em></p>
            <p><em>- Suzuki
                et al., 1989. An Introduction to Genetic Analysis 4th ed. W.H.
        Freeman p.704</em></p></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td><p class="style6"><font face="Arial, Helvetica, sans-serif">These non-selective,
          sampling-based effects are 
          one reason that it can be difficult to determine whether a particular
          trait is adaptive or not. It really depends upon the history of the
          population.</font></p>
      <p align="left" class="style6"><font face="Arial, Helvetica, sans-serif">The
              end result of founder effects, bottlenecks, and <a href="http://www.talkorigins.org/faqs/genetic-drift.html" target="_blank">genetic
          dr</a><a href="http://www.talkorigins.org/faqs/genetic-drift.html" target="_blank">ift</a> is that certain traits can be over-represented in a population by chance.&nbsp; </font></p>      
    </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td><hr size="1" noshade="noshade" /></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0" bgcolor="#CCCCCC">
    <tr>
        <td width="80" bgcolor="#990000"><div align="center">
          <p><span class="white">Questions
          to answer</span>          </p>
          <p>&nbsp;</p>
          <p><br />
          </p>
        </div></td>
        <td width="640" bgcolor="#FFFFFF"><font face="Arial, Helvetica, sans-serif">&nbsp; </font>
            <ol>
              <li>Consider the various ways that the
                individuals that fail to pass through a bottleneck might
                differ from those that do; how many kinds of &quot;reasons&quot; can
                you identify?</li>
              <li>Based on the Java Genetic Drift applet (below)
                - is it possible for an allele to go from 5% to 100%
                of the population simply by genetic drift?</li>
              <li>How does selection act to limit the
                effects of genetic drift? </li>
              <li>How might a genetic bottleneck
                 lead to a population with a higher frequency of a
                deleterious trait than was present in the population
                before the bottleneck?&nbsp;</li>
              <li>Assume that all members of a population
                that pass through a bottleneck have a deleterious trait;
                can the population survive and, if so, how would selection
                act on the population after the bottleneck? </li>
              <li>What is the difference between selection and
                  drift?&nbsp; </li>
              <li>How is it&nbsp;that drift can be quantified,
                  but in any particular experiment, not predicted?</li>
            </ol></td>
    </tr>
    <tr>
        <td bgcolor="#0000FF"><div align="center"><span class="whoe1"><span class="white">Questions
                        to ponder</span></span><br />
        </div></td>
        <td bgcolor="#FFFFFF"><ul>
            <li class="style17">What limits the &quot;size&quot; of
                the founder effect or a bottleneck effect? </li>
            <li class="style17">Does passing through a
                    bottleneck improve or hamper a population's chances for evolutionary
                success (that is, avoiding extinction)?</li>
        </ul></td>
    </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><hr size="1" noshade="noshade" /></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td width="54"><a href="https://solarsystem.colorado.edu/conceptInventories/external/bioliteracy.php" target="_blank"><img src="../../BioFun-Support/AllGraphics/eyeblnk1.gif" alt="eye" width="53" height="31" border="0" /></a></td>
    <td width="546"><div align="left">&nbsp;<span class="style17"><strong>before class</strong>&nbsp; [ <a href="http://www.youtube.com/watch?v=B5M_C8gBvYo" target="_blank">youtube introduction to the genetic drift applet</a>]</span></div></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><hr size="1" noshade="noshade" /></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td width="512"><p>Use 20 experiments for each population size (N)(from 10
      to 250) and <strong>plot
        the frequency of allele loss as a function of population size</strong> after
      100 generations.&nbsp;</p>
      <ul>
        <li>To do this you will need to determine the number of times that
          one or the other allele is lost for each 100 generations.&nbsp; Call
          that number L, and the frequency of allele loss <span class="style19">f</span><span class="style20"><sub>L</sub></span>.
          &nbsp; Then&nbsp;<span class="style19">f</span><span class="style20"><sub>L</sub></span> =
          L/20.&nbsp;&nbsp;&nbsp;</li>
        <li><strong>NOW:&nbsp; go to the BeSocratic Activity on Genetic Drift
                &amp; Bottlenecks</strong></li>
      </ul></td>
    <td width="10"><p>&nbsp;</p></td>
    <td width="178"><p><a href="http://darwin.eeb.uconn.edu/simulations/jdk1.0/drift.html" target="_blank">Use the <strong>Java
      Genetic Drift applet </strong>to answer these questions:&nbsp; </a></p>
      <p><font face="Arial, Helvetica, sans-serif"><strong>Note</strong>: in  the
        applet population, there is no selection, just sampling effects.&nbsp;</font></p></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td>replace with revised beSocratic activity</td>
  </tr>
</table>
<table width="720
" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td><hr />
    </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
      <td width="25"><font face="Arial, Helvetica, sans-serif"><b><a href="Topic1G_Evo.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="arrow" width="24" height="24" border="0" /></a></b></font></td>
    <td width="25"><div align="left"><a href="#top"><font face="Arial, Helvetica, sans-serif" size="-2"> <img src="../../BioFun-Support/AllGraphics/ArrowUP.GIF" width="24" height="24" border="0" /></font></a></div></td>
    <td width="633"><div align="right"><font size="-2" face="Arial, Helvetica, sans-serif"> modified 

          <!-- #BeginDate format:En2 -->10-May-2014<!-- #EndDate -->
    </font></div></td>
    <td width="9">&nbsp;</td>
    <td width="28"><div align="right"><strong><a href="Topic1I_Evo.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowRIGHT.gif" alt="" width="24" height="24" border="0" /></a></strong></div></td>
  </tr>
</table>

<script type='text/javascript'>var _hl=_hl||{};_hl.site='2622';(function(){var hl=document.createElement('script');hl.type='text/javascript';hl.async=true;hl.src=document.location.protocol+'//highlighter.com/webscript/v1/js/highlighter.js';var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(hl,s);})();</script>


</body>
</html>

