<!doctype html public "-//w3c//dtd html 4.0 transitional//en"><html><head>   <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">   <meta name="Content-Type" content="text/html; charset=iso-8859-1">   <meta name="description" content="Cell Biology & Development (with a little fun included).">   <meta name="keywords" content="Klymkowsky, Boulder, Xenopus, intermediate filaments, cell adhesion, Wnt signaling, plakoglobin">   <meta name="GENERATOR" content="Mozilla/4.75 (Macintosh; U; PPC) [Netscape]">   <meta name="Description" content="Cell Biology  &  Development (with a little fun included).">   <meta name="KeyWords" content="Klymkowsky, Boulder, Xenopus, intermediate filaments, cell adhesion, Wnt signaling, plakoglobin">   <title>Biofundamentals - Peptide bonds and polypeptides</title><script language="JavaScript"><!--function MM_preloadImages() { //v3.0  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}}function MM_swapImgRestore() { //v3.0  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;}function MM_findObj(n, d) { //v4.01  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);  if(!x && d.getElementById) x=d.getElementById(n); return x;}function MM_swapImage() { //v3.0  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}}//--></script></head><style type="text/css"><!--a:link {text-decoration: none}a:visited {text-decoration: none}a:active {text-decoration: none}a:hover {text-decoration: underline}--></style><body bgcolor="#FCFDFF" text="#000000" link="#0000FF" vlink="#0066FF" alink="#0066FF" onLoad="MM_preloadImages('AllGraphics/DLMovie.gif','AllGraphics/proline2.gif')"><table width="600" border="0" cellpadding="0">  <tr>     <td width="24"> <div align="left"><a href="#TOP"></a><a href="Topic2-8_Photosyn.htm" target="_self"><img src="CommonGraphics/ArrowLEFT.GIF" width="24" height="24" border="0"></a><font face="Arial, Helvetica, sans-serif" size="-1"><br>        </font></div></td>    <td width="20"><font face="Arial, Helvetica, sans-serif"><font color="#006600"><b><a name="top"></a></b></font></font></td>    <td width="548"><font face="Arial, Helvetica, sans-serif"><font color="#006600"><b>Peptide       bonds and polypeptides</b></font></font></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>       <hr>    </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="502"> <p><font face="Arial, Helvetica, sans-serif">We have already         mentioned <font color="#990000"><strong>proteins</strong></font>, since         there is almost no biological process that does not rely on them. </font></p>      <p><font face="Arial, Helvetica, sans-serif">Proteins derive their name         from the ancient, but minor, Greek sea-god <a href="http://www.olympian-foundation.org/proteus.htm" target="_blank"><b>Proteus         </b></a> who, like your typical sea-god, could change shape. The name         acknowledges the many different functions and properties of proteins.</font></p>      <p><font face="Arial, Helvetica, sans-serif">Proteins can act as catalysts         and regulators of chemical reactions. They control the expression of <font color="#990000"><strong>genes</strong></font>,         how genes respond to internal and external signals, and the replication         of the genetic material.</font></p>      <p><font face="Arial, Helvetica, sans-serif">They can be structural components,         determining both the shape and mechanical properties of cells and tissues.         </font></p>      <p><font face="Arial, Helvetica, sans-serif">They can be motors, responsible         for movements within cells, the movement of cells, tissues and the organism         as a whole. </font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <hr> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="376"> <p><font face="Arial, Helvetica, sans-serif">Proteins are         composed of <font color="#990000"><strong><img src="AllGraphics/alpha.gif" width="11" height="9">-amino         acids</strong></font> linked together by <strong><font color="#990000">peptide</font></strong>         <font color="#990000"><strong>bonds</strong></font> into <b><font color="#990000">polypeptide</font></b>         chains. </font></p>      <p><font face="Arial, Helvetica, sans-serif">An amino acid is characterized         by an <font color="#990000"><strong>amino </strong><font color="#000000">group</font></font>         (<strong>-NH<sub>2</sub></strong>) and a <font color="#990000"><strong>carboxylic         acid</strong></font> group (<strong>-COOH</strong>) linked to a carbon,         known as the <font color="#990000"><strong><img src="AllGraphics/alpha.gif" width="11" height="9"></strong></font>-carbon.         </font></p>      </td>    <td width="31">&nbsp;</td>    <td width="185">       <div align="right"> <img src="AllGraphics/AminoAcid.jpg" width="182" height="120"></div>    </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <p><font face="Arial, Helvetica, sans-serif">Also attached to the <font color="#990000"><strong><img src="AllGraphics/alpha.gif" width="11" height="9"></strong></font>-carbon         is a H and various '<font color="#990000"><strong>R-groups</strong></font>'         or '<font color="#990000"><strong>side-chains</strong></font>'. </font></p>      <p><font face="Arial, Helvetica, sans-serif">The four groups attached to         the </font> <font face="Arial, Helvetica, sans-serif"> <font color="#990000"><strong><img src="AllGraphics/alpha.gif" width="11" height="9"></strong></font>-carbon         are arranged at the vertices of a <font color="#990000"><strong>tetrahedron</strong></font>.         </font></p>      <p><font face="Arial, Helvetica, sans-serif">If the four groups are different         from one another, as they are in all amino acids except glycine, there         are two possible <font color="#990000"><strong>stereoisomers</strong></font>,         which are known as <font color="#990000"><strong>enantiomers.</strong></font></font></p></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="249"><a href="http://biomedia.bio.purdue.edu/IML/Amino/html/stereoisomers.html" target="_blank" onMouseOver="MM_swapImage('Image12','','AllGraphics/DLMovie.gif',1)" onMouseOut="MM_swapImgRestore()"><img src="AllGraphics/DLAminoAcids.jpg" name="Image12" width="248" height="128" border="0"></a>     </td>    <td width="12">&nbsp;</td>    <td width="331"><p><font face="Arial, Helvetica, sans-serif">These enantiomers         are mirror images of one another and are termed the L- and D- forms. </font></p>      <p><font face="Arial, Helvetica, sans-serif">O</font><font face="Arial, Helvetica, sans-serif">nly         L-type amino acids are found in proteins, even though there is <strong><a href="http://www.bio.puc.cl/pap03.htm" target="_blank">no         theoretical reason</a></strong> that proteins cannot be made of both types         of amino acids.</font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <hr> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td><font face="Arial, Helvetica, sans-serif">D-type amino acids do occur       in nature. They are found in biomolecules, such as the antibiotic <font color="#990000"><strong>gramicidin,       </strong></font>which is composed of alternating L-and D-type amino acids.       </font> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="280"> <object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,29,0" width="280" height="360">        <param name="movie" value="AllGraphics/AminoAcidViewer.swf">        <param name="quality" value="high">        <embed src="AllGraphics/AminoAcidViewer.swf" quality="high" pluginspage="http://www.macromedia.com/go/getflashplayer" type="application/x-shockwave-flash" width="280" height="360"></embed></object>     </td>    <td width="12">&nbsp; </td>    <td width="300"><p><font face="Arial, Helvetica, sans-serif">Even though there         are hundreds of different amino acids known, only 19 amino acids and one         <font color="#990000"><strong>imino acid</strong></font>, proline, are         found in proteins. </font></p>      <p><font face="Arial, Helvetica, sans-serif">These amino acids differ in         their R-groups. </font></p>      <p><font face="Arial, Helvetica, sans-serif">Some of these R-groups are         highly hydrophobic, some are hydrophilic, some are positively or negatively         charged. </font></p>      <p><font face="Arial, Helvetica, sans-serif">The different R-groups provide         proteins with a range of chemical properties.</font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="510" bgcolor="#CCCCCC"> <hr /> </td>    <td width="84" bgcolor="#CCCCCC"><div align="right"><a href="Questions/Block-3.htm#eight" target="_blank"><img src="CommonGraphics/questions.gif" width="84" height="15" border="0" /></a></div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <p><font face="Arial, Helvetica, sans-serif"><strong>Building a polypeptide</strong></font>       </p>      <p><font face="Arial, Helvetica, sans-serif"><strong><font color="#990000">Polymers</font></strong>         are chains of subunits, <font color="#990000"><strong>monomers</strong></font>,         linked together by chemical bonds. </font></p>      <p><font face="Arial, Helvetica, sans-serif">The bond between two amino         acids is known as a <font color="#990000"><strong>peptide bond</strong><font color="#000000">.         Two amino acids, joined together by a peptide bond, are known as a <font color="#990000"><strong>dipeptide.</strong></font></font></font></font></p>      <p><font face="Arial, Helvetica, sans-serif"><font color="#990000"><font color="#000000">An         amino acid polymer is known as a </font><strong>polypeptide</strong></font>.         This is a complex polymer, composed of 20 different possible monomers         that can, in theory, be linked together in any imaginable order. </font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="277"><p><font face="Arial, Helvetica, sans-serif">The formation         of a peptide bond involves a condensation / dehydration reaction. Two         molecules become one and water is released.</font></p>      </td>    <td width="14">&nbsp;</td>    <td width="301"> <p>        <object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,29,0" width="300" height="160">          <param name="movie" value="AllGraphics/peptideBond.swf">          <param name="quality" value="high">          <embed src="AllGraphics/peptideBond.swf" quality="high" pluginspage="http://www.macromedia.com/go/getflashplayer" type="application/x-shockwave-flash" width="300" height="160"></embed></object>        <font face="Arial, Helvetica, sans-serif"> </font></p></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td><hr> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <font face="Arial, Helvetica, sans-serif">When a polypeptide chain is       being synthesized, new amino acid subunits are attached to the C- or carboxy       terminus of the existing chain. </font>       <p><font face="Arial, Helvetica, sans-serif">This generates an unbranched,         linear polymer. </font></p>      <p><font face="Arial, Helvetica, sans-serif">The peptide bond has a number         of characteristics that are critical in determining how polypeptides behave.         Although drawn as a single bond, the peptide bond behaves more like double         bond. </font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="300">       <object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,29,0" width="300" height="160">        <param name="movie" value="AllGraphics/rotatingBonds.swf">        <param name="quality" value="high">        <embed src="AllGraphics/rotatingBonds.swf" quality="high" pluginspage="http://www.macromedia.com/go/getflashplayer" type="application/x-shockwave-flash" width="300" height="160"></embed></object>     </td>    <td width="12" bgcolor="#CCCCCC">&nbsp;</td>    <td width="280"><p><font face="Arial, Helvetica, sans-serif">While there is         free rotation around a single bond rotation, around a peptide bond is         constrained. </font></p>      <p><font face="Arial, Helvetica, sans-serif">Moreover, the <font color="#990000"><strong>carbonyl         oxygen</strong></font> (<strong>-C=O</strong>) acts as a H-bond acceptor,         while the <font color="#990000"><strong>amino hydrogen</strong></font>         (<strong>-N-H</strong>) acts as a H-bond donor. </font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <hr> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="283"> <table width="100%" border="0">        <tr>          <td width="64%"><div align="right"><font face="Arial, Helvetica, sans-serif"><strong><font color="#990000">Proline</font></strong>               is not an amino acid,</font><font face="Arial, Helvetica, sans-serif">               but rather an <font color="#990000"><strong>imino acid</strong></font>.</font></div></td>          <td width="36%"><div align="right"><a href="#" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_swapImage('proline','','AllGraphics/proline2.gif',1)"><img src="AllGraphics/proline1.gif" name="proline" width="78" height="53" border="0"></a></div></td>        </tr>      </table>      <p align="left"><font face="Arial, Helvetica, sans-serif">It makes a rather         different type of peptide bond which is not nearly as constrained as a         typical peptide bond. </font></p>      </td>    <td width="12" bgcolor="#CCCCCC">&nbsp;</td>    <td width="301"><p align="right"><font face="Arial, Helvetica, sans-serif"><strong><font color="#990000">         <object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=5,0,0,0" width="300" height="180">          <param name=movie value="AllGraphics/proline.swf">          <param name=quality value=high>          <embed src="AllGraphics/proline.swf" quality=high pluginspage="http://www.macromedia.com/shockwave/download/index.cgi?P1_Prod_Version=ShockwaveFlash" type="application/x-shockwave-flash" width="300" height="180">           </embed> </object>        </font></strong></font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="214"><p><img src="AllGraphics/prolineKink.jpg" width="200" height="185"></p></td>    <td width="12">&nbsp;</td>    <td width="366"><p><font face="Arial, Helvetica, sans-serif">Proline peptide         bonds are found in the <font color="#990000"><strong>cis</strong></font>         configuration ~100 times as often as those between other amino acids.</font></p>      <p><font face="Arial, Helvetica, sans-serif">There is <strong>no H-bond         donor</strong> in the proline peptide bond </font><font face="Arial, Helvetica, sans-serif">and         the presence of proline leads to a bend or kink in the polypeptide chain.</font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="510" bgcolor="#CCCCCC"> <hr /> </td>    <td width="84" bgcolor="#CCCCCC"><div align="right"><a href="Questions/Block-3.htm#eight" target="_blank"><img src="CommonGraphics/questions.gif" width="84" height="15" border="0" /></a></div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <p><font face="Arial, Helvetica, sans-serif"><strong>Water, polypeptide         synthesis and folding</strong></font></p>      <p><font face="Arial, Helvetica, sans-serif">Were it </font><font face="Arial, Helvetica, sans-serif">not         for the various R-groups polypeptides would assume an extended configuration         in water. An example is <a href="http://www.nature.com/cgi-taf/DynaPage.taf?file=/nsb/journal/v5/n8/full/nsb0898_675.html" target="_blank"><font color="#0000CC"><strong>polyglycine</strong></font></a>.         </font></p>      <p><font face="Arial, Helvetica, sans-serif">H-bond donors and acceptor         groups in polypeptide backbone would form H-bonds with each other and         with water molecules. </font></p>      <p><font face="Arial, Helvetica, sans-serif">More typical polypeptides have         all of the different R-groups in various proportions. Many of these R-groups         are <strong> <a href="http://www.russell.embl-heidelberg.de/aas/hydrophobic.html" target="_blank">highly         hydrophobic</a></strong>. Their presence makes an extended configuration         energetically unfavorable. </font></p>      <p><font face="Arial, Helvetica, sans-serif">Very much like the process         by which lipids self-assemble to form micelles and bilayers, a polypeptide         in aqueous solution will collapse onto itself in order to remove as many         of these <font color="#990000"><strong>hydrophobic residues</strong></font>         from contact with water as possible. </font></p></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="379"><font face="Arial, Helvetica, sans-serif">Protein folding       is one of the most complicated computational problems in modern science.       </font> </td>    <td width="13">&nbsp;</td>    <td width="200"><div align="right"><font face="Arial, Helvetica, sans-serif"><a href="http://folding.stanford.edu/" target="_blank"><img src="AllGraphics/foldingAThone.gif" width="200" height="49" border="0"></a>         </font></div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td><hr size="1"> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td><p><font face="Arial, Helvetica, sans-serif">It is a basic assumption         of structural biology that the final folded state of a polypeptide is         determined by the sequence of amino acids along its length, and that this         final structure is the state of lowest energy</font></p>      <p><font face="Arial, Helvetica, sans-serif">This sequence of amino acids         is known as the <font color="#990000"> <strong>primary structure</strong></font>         of the polypeptide.</font></p>      <p><font face="Arial, Helvetica, sans-serif">We write the amino acid sequence         of a polypeptide from its <strong>N-</strong> or <font color="#990000"><strong>amino         terminus</strong></font> to its <strong>C-</strong> or <font color="#990000"><strong>carboxyl         terminus</strong></font>, with the N-terminus to the left and the C-terminus         to the right.</font> </p></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td><hr size="1"> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="382"> <p><font face="Arial, Helvetica, sans-serif">Proteins structure         is commonly presented in a <font color="#990000"><strong>hierarchical</strong></font>         manner. </font></p>      <p><font face="Arial, Helvetica, sans-serif">While this is an over-simplification,         it is a good place to start. </font></p>      <p><font face="Arial, Helvetica, sans-serif">We will consider the actual         process of protein synthesis, known as <font color="#990000"><strong>translation</strong></font>,         in the next reading. </font></p>      <p><font face="Arial, Helvetica, sans-serif">As it is being synthesized,         the polypeptide chain begins to fold. </font></p>      </td>    <td width="12" bgcolor="#CCCCCC">       <div align="right"></div>    </td>    <td width="201"><object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=5,0,0,0" width="200" height="200">        <param name=movie value="AllGraphics/proteinstructure.swf">        <param name=quality value=high>        <embed src="AllGraphics/proteinstructure.swf" quality=high pluginspage="http://www.macromedia.com/shockwave/download/index.cgi?P1_Prod_Version=ShockwaveFlash" type="application/x-shockwave-flash" width="200" height="200">        </embed>       </object></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="240"><p><img src="AllGraphics/FoldingLandscape.jpg" width="240" height="227">       </p>      </td>    <td width="12">&nbsp;</td>    <td width="340"><div align="right">         <p align="left"><font face="Arial, Helvetica, sans-serif">We can think           of the folding process as a walk across an <font color="#990000"><strong>energy           landscape</strong></font>. </font></p>        <p align="left"><font face="Arial, Helvetica, sans-serif">The goal is           to find the lowest point in the landscape, the energy minimum of the           system. This is generally assumed to be the <font color="#990000"><strong>native</strong></font>           or functional state of the polypeptide.</font></p>        <p><font face="Arial, Helvetica, sans-serif"> The first step is the movement           of hydrophobic R-groups out of contact with water. </font></p>        <p><font face="Arial, Helvetica, sans-serif">This drives the collapse           of the polypeptide into a compact and dynamic '<font color="#990000"><strong>molten           globule</strong></font>'. </font></p>      </div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="363"><p><font face="Arial, Helvetica, sans-serif">The path to the         native state is not necessarily a smooth or unambiguous one. </font></p>      <p><font face="Arial, Helvetica, sans-serif">The folding polypeptide can         get 'stuck' in a local energy minimum. </font></p>      <p><font face="Arial, Helvetica, sans-serif">If a polypeptide get stuck,         there are mechanisms to unfold it and let it 'try' again to reach its         native state. </font></p> </td>    <td width="12">&nbsp;</td>    <td width="221"><p><font face="Arial, Helvetica, sans-serif">         <object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,29,0" width="220" height="160">          <param name="movie" value="AllGraphics/landscape.swf">          <param name="quality" value="high">          <embed src="AllGraphics/landscape.swf" quality="high" pluginspage="http://www.macromedia.com/go/getflashplayer" type="application/x-shockwave-flash" width="220" height="160"></embed></object>        </font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <font face="Arial, Helvetica, sans-serif">This process partial unfolding       is carried out by proteins known as <font color="#990000"><strong>chaperones</strong></font>.       We will return to the mechanism of chaperone action later. </font> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="510" bgcolor="#CCCCCC"> <hr /> </td>    <td width="84" bgcolor="#CCCCCC"><div align="right"><a href="Questions/Block-3.htm#eight" target="_blank"><img src="CommonGraphics/questions.gif" width="84" height="15" border="0" /></a></div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <p><font face="Arial, Helvetica, sans-serif"><strong>Secondary structure         features</strong></font></p>      <p><font face="Arial, Helvetica, sans-serif">All polypeptides share a common         backbone structure made up of peptide bonds.</font><font face="Arial, Helvetica, sans-serif">It         is therefore not surprising that there are common patterns or <font color="#990000"><strong>motifs</strong></font>         in polypeptide folding.</font></p>      <p><font face="Arial, Helvetica, sans-serif">The first of these, the <img src="AllGraphics/alpha.gif" width="11" height="9">         helix, was discovered by <strong><a href="http://www.paulingexhibit.org/exhibit/alpha-helix.html" target="_blank">Linus         Pauling</a></strong> and Robert Corey, and reported in 1951. </font></p>      <p align="right"><font size="-2" face="Arial, Helvetica, sans-serif">Linus         at age 5.</font></p>      <p align="left"><font face="Arial, Helvetica, sans-serif">This was followed         shortly thereafter by the structure of the &szlig;-sheet. </font></p>      <p align="left"><font face="Arial, Helvetica, sans-serif">The forces that         drive the formation of the <img src="AllGraphics/alpha.gif" width="11" height="9">         helix and the &szlig;-sheet will be familiar. They are they same forces         that underlie water structure.</font></p></td>    <td>&nbsp;</td>    <td><a href="http://www.orst.edu/dept/lpi/lpbio/lpbio2.html" target="_blank"><img src="AllGraphics/youngLinus.jpg" width="103" height="280" border="0"></a></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="113"> <p><img src="AllGraphics/alphaHelix.jpg" width="113" height="200"><font face="Arial, Helvetica, sans-serif">         </font></p></td>    <td width="7">&nbsp;</td>    <td width="287"> <p><font face="Arial, Helvetica, sans-serif">In an <img src="AllGraphics/alpha.gif" width="11" height="9">-helix         </font><font face="Arial, Helvetica, sans-serif">all of the possible H-bonds         involving the peptide bond donor and acceptor groups are formed. This         stabilizes the structure. </font></p>      <p><font face="Arial, Helvetica, sans-serif">In an <img src="AllGraphics/alpha.gif" width="11" height="9">-helix,         the R-groups point outward from the long axis of the helix. </font><font face="Arial, Helvetica, sans-serif">         </font></p></td>    <td width="183"><div align="right"><img src="AllGraphics/axialhelix.jpg" width="180" height="191"></div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>       <hr>    </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <p><font face="Arial, Helvetica, sans-serif">In &szlig;-sheets, the H-bonds         between the peptide bond donor and acceptor groups are made between either         <font color="#990000"><strong>parallel </strong></font>or <font color="#990000">         <strong>anti-parallel</strong></font> polypeptide chains. </font></p>      <p align="right"><font size="-2" face="Arial, Helvetica, sans-serif">These         are anti-parallel &szlig;-sheets</font></p>      <p align="left"><font face="Arial, Helvetica, sans-serif">The R-groups point         out of and into the plane of the sheet. </font></p>      </td>    <td>&nbsp;</td>    <td><img src="AllGraphics/betasheet.gif" width="429" height="256"></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <hr> </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="103"> <b><font face="Arial, Helvetica, sans-serif">      <object classid="clsid:D27CDB6E-AE6D-11cf-96B8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=5,0,0,0" width="100" height="160">        <param name=movie value="AllGraphics/PaulingCorey.swf">        <param name=quality value=high>        <embed src="AllGraphics/PaulingCorey.swf" quality=high pluginspage="http://www.macromedia.com/shockwave/download/index.cgi?P1_Prod_Version=ShockwaveFlash" type="application/x-shockwave-flash" width="100" height="160">         </embed> </object>      </font></b> </td>    <td width="12">&nbsp;</td>    <td width="477"><p><font face="Arial, Helvetica, sans-serif">The pattern of         <img src="AllGraphics/alpha.gif" width="11" height="9">-helices, &szlig;-sheets         and connecting regions is known as the <font color="#990000"><strong>secondary         structure</strong></font> of the polypeptide.</font></p>      <p><font face="Arial, Helvetica, sans-serif">The key factor that determines         the specific details of polypeptide folding is the interaction between         R-groups, not only with one another but with water and dissolved ions.         </font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="247"><p><font face="Arial, Helvetica, sans-serif">In a water soluble         polypeptide, most but not necessarily all of the surface R-groups will         by hydrophilic.</font></p>      <p><font face="Arial, Helvetica, sans-serif"> Hydrophobic groups will tend         to be buried in the polypeptide's interior.</font></p>      <p><font face="Arial, Helvetica, sans-serif"> Some polypeptides are inserted         into membranes. In these polypeptides, hydrophobic R-groups on the surface         of the folded polypeptide will interact with the hydrophobic interior         of the lipid bilayer</font></p>      <b><font face="Arial, Helvetica, sans-serif"> </font></b></td>    <td width="44">&nbsp;</td>    <td width="301"><p><font face="Arial, Helvetica, sans-serif"><b><img src="AllGraphics/hydrophobicCore.gif" width="300" height="288"></b></font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td height="13"> <font face="Arial, Helvetica, sans-serif">Ionic interactions       between acidic and basic R-groups, van der Waals interactions and H-bonds       also play an important role in determining the <font color="#990000"><strong>tertiary</strong></font>       or three dimensional structure of the polypeptide. </font></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <hr></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="467" height="13"> <p><font face="Arial, Helvetica, sans-serif">A         <font color="#990000"> <strong>protein</strong></font> is a functional         entity. In many cases a protein is composed of multiple polypeptides.         </font></p>      <p><font face="Arial, Helvetica, sans-serif">A protein that contains multiple         polypeptides, whether of the same or different types, is said to have         a <font color="#990000"><strong>quaternary</strong></font> structure.         </font></p>      <p align="right"><font size="-2" face="Arial, Helvetica, sans-serif">A protein         with two polypeptide subunits.</font></p>      <p align="left"><font face="Arial, Helvetica, sans-serif">There are also         higher levels of polypeptide organization. For example, a specific polypeptide         may be a component for a number of different proteins. </font></p>      </td>    <td width="12">&nbsp;</td>    <td width="119"><img src="AllGraphics/multiSubunitProtein.gif" width="119" height="220"></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>&nbsp; </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td> <p><font face="Arial, Helvetica, sans-serif">The polypeptide may form         stable interactions and be an integral part of a single protein molecule         for its entire existence, </font></p>      <p><font face="Arial, Helvetica, sans-serif">Alternatively, the polypeptide         may actively exchange partners and be part of a number of a different         proteins during its lifetime. </font></p>      <p><font face="Arial, Helvetica, sans-serif">A polypeptide that takes part         in many dynamic interactions is part of a regulatory network. Such a polypeptide         can be said to have a <font color="#990000"><strong>quinary</strong></font>         structure. </font> </p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="510" bgcolor="#CCCCCC"> <hr /> </td>    <td width="84" bgcolor="#CCCCCC"><div align="right"><a href="Questions/Block-3.htm#eight" target="_blank"><img src="CommonGraphics/questions.gif" width="84" height="15" border="0" /></a></div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>       <p><font face="Arial, Helvetica, sans-serif"><b>Other factors in protein         structure</b></font></p>      <p><font face="Arial, Helvetica, sans-serif">In addition to polypeptides,         proteins can include non-amino acid-based components, known generically         as <b><font color="#990000">co-factors</font></b>. </font></p>      <p><font face="Arial, Helvetica, sans-serif">A protein minus its cofactors         is known as the <b><font color="#990000">apoprotein</font></b><font color="#000000">         or </font><b><font color="#990000">apoenzyme</font></b>, </font> </p>    </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="387"><p><font face="Arial, Helvetica, sans-serif">Together with         its cofactors, it is known as the <b><font color="#990000">holoprotein</font></b>         or <b><font color="#990000">holoenzyme</font></b>. </font></p>      <p><font face="Arial, Helvetica, sans-serif">A protein without its cofactors         is inactive and often unstable. </font> </p>      <p><font face="Arial, Helvetica, sans-serif">Cofactors can range in complexity         from a single metal atom to quite complex molecules, such as the vitamin         B<sub>12</sub></font></p>    </td>    <td width="16">&nbsp;</td>    <td width="189"><div align="right"><img src="AllGraphics/vitaminB12.gif" width="187" height="182"></div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td>       <hr>    </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="154"><img src="AllGraphics/CysteineKnot.jpg" width="154" height="260">     </td>    <td width="12">&nbsp;</td>    <td width="426"> <p><font face="Arial, Helvetica, sans-serif">Another important         stabilizing element in protein structure is the formation of <font color="#990000"><strong>disulfide         bonds</strong></font> between <font color="#990000"><strong>cysteine residues</strong></font>.</font>       </p>      <p><font face="Arial, Helvetica, sans-serif">These are strong covalent bonds.         They are generally formed once the polypeptide has folded correctly and         stabilize the final folded structure. </font></p>      <p><font face="Arial, Helvetica, sans-serif">If they form too early, however,         they can block the correct folding of the polypeptide and must be removed         for correct folding to occur.</font></p>      </td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="510" bgcolor="#CCCCCC"> <hr /> </td>    <td width="84" bgcolor="#CCCCCC"><div align="right"><a href="Questions/Block-3.htm#eight" target="_blank"><img src="CommonGraphics/questions.gif" width="84" height="15" border="0" /></a></div></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="262"> <strong><font color="#006600" size="-1" face="Arial, Helvetica, sans-serif">kAssess&#8482;       True Knowledge Profile</font></strong></td>  </tr></table><table width="600" border="0" cellpadding="0">  <tr>     <td width="35"> <div align="left"><a href="#TOP"><font face="Arial, Helvetica, sans-serif" size="-2"><img src="CommonGraphics/ArrowUP.GIF" width="24" height="24"></font></a></div></td>    <td width="416"> <div align="right">         <p align="left"><font size="-2" face="Arial, Helvetica, sans-serif">Check           the <a href="http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?db=Books" target="_blank"><strong>NCBI           BookShelf</strong></a> | 12 November 2002</font></p>      </div></td>    <td width="141"><div align="right"><a href="Topic3-2_Proteins.htm" target="_self"><img src="CommonGraphics/ArrowRIGHT.gif" width="24" height="24" border="0"></a></div></td>  </tr></table><font face="Book Antiqua Italic"><b><font size=+1></font></b></font><font face="Book Antiqua Italic"></font> </body></html>