<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html>
<head>
<meta name="generator" content="HTML Tidy for Linux (vers 25 March 2009), see www.w3.org">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="Content-Type" content="text/html; charset=iso-8859-1">
<meta name="description" content="Interactive Introductory / Foundational Evolution and Molecular Biology">
<meta name="GENERATOR" content="Mozilla/4.75 (Macintosh; U; PPC) [Netscape]">
<title>Biofundamentals @ UC Boulder- Protein Assembly &amp; turnover</title>

<script language="JavaScript" type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
//-->
</script>
<style type="text/css">
<!--
a:link {
        text-decoration: none
}
a:visited {
        text-decoration: none
}
a:active {
        text-decoration: none
}
a:hover {
        text-decoration: underline
}
.style1 {
        font-family: Arial, Helvetica, sans-serif;
        font-size: x-small;
}
.style2 {
        font-family: Arial, Helvetica, sans-serif
}
.style3 {
        color: #990000;
        font-weight: bold;
}
body, td, th {
        font-family: Arial, Helvetica, sans-serif;
}
.style4 {
        font-size: x-small
}
.white {
        color: #FFF;
}
-->
</style>

<style type="text/css">
 body {
  background-color: #FCFDFF;
  color: #000000;
 }
 :link { color: #0000FF }
 :visited { color: #0066FF }
 :active { color: #0066FF }
 div.c17 {font-family: Arial, Helvetica, sans-serif; font-size: 64%; text-align: right}
 div.c16 {text-align: left}
 span.c15 {font-family: Arial, Helvetica, sans-serif; font-size: 64%}
 table.c14 {background-color: #CCCCCC}
 td.c13 {background-color: #0000FF}
 td.c12 {background-color: #990000}
 div.c11 {text-align: center}
 div.c10 {font-family: Arial, Helvetica, sans-serif; text-align: center}
 p.c9 {font-family: Arial, Helvetica, sans-serif; text-align: right}
 span.c8 {color: #000000}
 p.c7 {text-align: left}
 p.c6 {font-family: Arial, Helvetica, sans-serif; text-align: left}
 span.c5 {font-family: Arial, Helvetica, sans-serif}
 td.c4 {background-color: #FFFFFF}
 p.c3 {font-family: Arial, Helvetica, sans-serif}
 div.c2 {text-align: right}
 span.c1 {color: #000000; font-family: Arial, Helvetica, sans-serif}
</style>
</head>
<body onload="MM_preloadImages('../../Biofundamentals/lectureNotes/graphics/week5graphics/prpsc.gif')">
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="667"><strong><span class="c1">23. Protein assembly &amp; tunover</span></strong></td>
<td width="24"><a href="Topic3-6_Making Proteins.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="" width="24" height="24" border="0"></a></td>
<td width="29">
<div class="c2"><a href="Topic3-8_Protein Activity.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowRIGHT.gif" alt="arrow" width="24" height="24" border="0"></a></div>
</td>
</tr>
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c3">New amino acids are added to the polypeptide's C-terminus, while the N-terminus moves through a tunnel within the ribosome; about 15 amino acids are added per second.</p>
<p class="c3">The ribosomal tunnel is approximately 10 nm long and 1.5 nm in diameter; it is narrow enough to block the folding of the newly synthesized polypeptide chain.</p>
</td>
<td> </td>
<td><img src="../../BioFun-Support/wordles/page20.jpg" width="320" height="160" alt="page 19"></td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p>Translation, like most all processes that occur within the cell, is a <a href="http://en.wikipedia.org/wiki/Stochastic" target="_blank">stochastic</a> process, meaning that it is based on random collisions between molecules. In the specific case of translation, the association of the mRNA with the ribosomal components occurs stochastically; similarly, the addition of a new amino acid depends on the collision of the appropriate amino acid-charged, tRNAs with the RNA-ribosome complex.</p>
<p>Since there are many "inappropriate" amino-acid charged tRNAs in the cytoplasm, the ribosomal complex must be able to productively bind only the tRNA the mRNA specifies, that is the tRNA with the right anticodon.  This enables its attached amino acid can interact productively to add the amino acid to the growing polypeptide chain.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="22%"><img src="../../BioFun-Support/AllGraphics/peptidePath.gif" width="153" height="164" alt="** PLEASE DESCRIBE THIS IMAGE **"></td>
<td class="c4" width="1%"> </td>
<td width="77%">
<p class="c3">In most illustrations of polypeptide synthesis, you rarely see this fact illustrated.</p>
<p><span class="c5">As the polypeptide emerges from the tunnel, it encounters the crowded cytoplasmic environment; at the same time it begins</span> <span class="c5">to fold. </span></p>
<p class="c3">As it folds, the polypeptide needs to avoid non-specific interactions with other cellular components. If it is part of a multisubunit protein, it must "find" its partner polypeptides, again, a stochastic process.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td><iframe width="720" height="406" src="http://www.youtube.com/embed/i8rGTYQ6oZ8" frameborder="0"></iframe></td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="578">
<p class="c3">If the polypeptide does not fold correctly, it will not function correctly and may damage the cell.</p>
<p class="c3">A number of degenerative neurological disorders are due, at least in part, to the accumulation of misfolded polypeptides (see below).</p>
<p class="c3">A number of enzymes, known generically as chaperones, associate with the nascent polypeptide as it emerges from the ribosome.</p>
<p class="c3">Chaperones facilitate the proper folding of the polypeptide, primarily by suppressing or unfolding 'incorrect' structures (perhaps you can guess how a chaperone "knows" something bad is going on).</p>
<p class="c3">Chaperones bind to, and stabilize polypeptides prior to their final assembly into multisubunit complexes, the final active protein</p>
</td>
<td class="c4" width="10"> </td>
<td width="114">
<div class="c2"><img src="../../BioFun-Support/AllGraphics/ChaperoneNascentPolypeptide.jpg" width="113" height="300" alt="** PLEASE DESCRIBE THIS IMAGE **"></div>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="362"><iframe width="358" height="260" src="http://www.youtube.com/embed/swEc_sUVz5I?rel=0" frameborder="0"></iframe></td>
<td width="339"> </td>
<td width="19"><iframe width="358" height="260" src="http://www.youtube.com/embed/gFcp2Xpd29I?rel=0" frameborder="0"></iframe></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="80">
<p><img src="../../BioFun-Support/AllGraphics/RefoldingChaperone.jpg" width="80" height="260" alt="** PLEASE DESCRIBE THIS IMAGE **"></p>
</td>
<td class="c4" width="14"> </td>
<td width="566">
<div class="c2">
<p class="c6">One class of chaperones are known as heat shock proteins.</p>
<p class="c6">These proteins recognize unfolded polypeptides - protein unfolding occurs with increasing frequency at higher temperatures.</p>
<p class="c6">Heat-shock chaperones couple ATP hydrolysis reactions with polypeptide unfolding reactions to unfold misfolded polypeptides.  They then release the unfolded polypeptide or protein, giving it another chance to refold correctly, on its own. </p>
<p class="c6">How do chaperones recognize unfolded or abnormally folded proteins? They look for hydrophobic regions on their surface.</p>
<p class="c6">Normal soluble proteins do not have such regions and the presence of such domains can lead to non-specific protein-protein aggregates.</p>
</div>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td><iframe width="720" height="540" src="http://www.youtube.com/embed/b39698t750c" frameborder="0"></iframe></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c3"><strong>Protein degradation: </strong></p>
<p><span class="c5">If a protein does not fold correctly, it is generally degraded by enzymes known as proteases, enzymes that cleave proteins. This involves forming a complex with the protease.</span> <span class="c5">Proteases cleave peptide bonds via a hydrolysis reaction.</span> The hydrolysis of a protein is a thermodynamically favorable reaction. </p>
<p>It is the case that proteins differ in the probability that they will form a productive complex with a <span class="c5">protease. In</span> this case, a productive interaction means that the protein will be degraded.  <span class="c5">The process of degradation is known as turnover, in part because the amino acids released can be reused by the cell.</span></p>
<p>The end result of the processes of translation (protein synthesis), folding, and degradation is that each protein in a cell has a half-life. If we had a population of molecules, the half-life is the time at which we would expect 50% of the those molecules <span class="c5">present at time = 0 to have been degraded through interactions with proteases. </span></p>
<p class="c3">There are a number of mechanisms involved in the turnover of proteins.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="401">
<div class="c2"><a href="http://www.biocarta.com/pathfiles/proteasomePathway.asp" target="_blank"><img src="../../BioFun-Support/AllGraphics/proteosome.jpg" alt="degradation" width="400" height="298" border="0"></a></div>
</td>
<td width="12"> </td>
<td width="287">
<div class="c2">
<p class="c6">A common one involves a macromolecular machine, the proteosome.</p>
<p class="c7"><span class="c5">The proteosome is a multienzyme complex that degrades polypeptides. The proteases can be divided into two generic classes.</span> <span class="c5">Those that hydrolyze polypeptides from one end or the other, to release one or twp amino acid at a time, are known as exoproteases. </span></p>
<p class="c7">Please note (in the video below), the unrealistic components.</p>
</div>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td><iframe width="720" height="540" src="http://www.youtube.com/embed/4DMqnfrzpKg" frameborder="0"></iframe>
<p class="c7"> </p>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c3">Proteases that cleave a polypeptide chain internally are known as endoproteases - they generate two polypeptides. </p>
<p>Whether a protein is degraded depends upon how it interacts with other molecules, particularly chaperones and proteosomes, and other proteases [a mathematical analysis of protein (and RNA) half-life is beyond the scope of this course, but you can read about it <a href="http://en.wikipedia.org/wiki/Protein_turnover">here</a> and <a href="http://www.ncbi.nlm.nih.gov/pubmed/15596551" target="_blank">here</a> if you wish.]</p>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c7"><span class="c5"><strong>Diseases of folding and misfolding:</strong> If the functional protein is in its native state</span>, a dysfunctional misfolded protein is said to be denatured.</p>
<p class="c6">It does not take much of a perturbation to unfold or denature most proteins. In fact, under normal conditions, proteins often become partially denatured. A number of diseases have been associated with <a href="http://www.faseb.org/opar/protfold/protein.html" target="_blank">protein misfolding</a><strong>.</strong></p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="549">
<p><span class="c5">Kuru</span> was among the first of these to be identified. Beginning in the 1950s, <a href="http://www.xrefer.com/entry/494426" target="_blank">D. Carleton Gadjusek</a> studied a neurological disorder common among the Fore people of New Guinea.</p>
<p class="c3">The symptoms of <a href="http://www.xviral.co.uk/disease/kuru.htm" target="_blank">kuru</a>, which means "trembling with fear", are similar to those of <a href="http://www.niddk.nih.gov/intram/people/wickner/prions.htm" target="_blank">scrapie</a>, a disease of sheep, and <a href="http://www.emedicine.com/med/topic1248.htm" target="_blank">variant Creutzfeld-Jakob disease</a> (vCJD) in humans.</p>
</td>
<td width="10"> </td>
<td width="101">
<div class="c2"><a href="http://www.nobel.se/medicine/laureates/1976/gajdusek-lecture.html" target="_blank"><img src="../../BioFun-Support/AllGraphics/gajdusek.jpg" width="100" height="141" border="0" alt="** PLEASE DESCRIBE THIS IMAGE **"></a></div>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c3">Among the Fore people, kuru was linked to the ritual eating of the dead. Since this practice has ended, the disease has disappeared.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="100"><a href="http://www.nobel.se/medicine/laureates/1997/illpres/" target="_blank"><img src="../../BioFun-Support/AllGraphics/prusiner.jpg" width="100" height="141" border="0" alt="** PLEASE DESCRIBE THIS IMAGE **"></a></td>
<td width="10"> </td>
<td width="590">
<p class="c3"><strong>Prions and protein folding:</strong></p>
<p><span class="c5">The cause of kuru, scrapie and vCJD appears to be the presence of an abnormal form of a normal protein, known as a prion</span>. We can think of prions as an anti-chaperone.</p>
<p class="c3">The idea of proteins as infectious agents was championed by Stan Prusiner.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="557"><span class="c5">The protein responsible for kuru and scrapie is known as PrP<sup>c</sup><strong><span class="c8">. </span></strong></span> It normally exists in a largely α-helical form.  There is a second abnormal form of the protein, PrP<sup>sc</sup> for scrapie; it is composed primarily of β-sheet. </td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="415">
<p class="c3">The two polypeptides have the same primary sequence.  <span class="c8">PrP<sup>sc</sup></span> acts as an anti-chaperone, catalyzing the transformation of <span class="c8">PrP<sup>c</sup></span> into <span class="c1">PrP<sup>sc</sup></span>.</p>
<p><span class="c5">Once initiated, this leads to a chain reaction</span> and <span class="c1">PrP<sup>sc</sup></span> accumulation. <span class="c1">As it accumulates, PrPsc assembles</span> into rod-shaped aggregates that appear to damage cells.</p>
</td>
<td width="10"> </td>
<td width="277">
<div class="c2"><img src="../../BioFun-Support/AllGraphics/prionForms.jpg" width="276" height="189" alt="** PLEASE DESCRIBE THIS IMAGE **"></div>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="360"><img src="../../BioFun-Support/AllGraphics/prion.jpg" width="360" height="200" alt="** PLEASE DESCRIBE THIS IMAGE **"></td>
<td class="c4" width="13"> </td>
<td width="347">
<p class="c3">This process occurs within the cells of the central nervous system, and leads to severe neurological defects.</p>
<p class="c3">There is no natural defense, since the protein responsible is a normal protein.</p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="528">
<div class="c2">
<p class="c7"><strong>Disease transmission: </strong>When the Fore ate the brains of their beloved ancestors, they inadvertently introduced the <span class="c8">PrP<sup>sc</sup></span> protein into their bodies.</p>
<p class="c7">Genetic studies indicate that early humans evolved resistance to prion diseases, suggesting that cannibalism might have been an important selective factor during human evolution.<span class="style1"> </span></p>
</div>
</td>
<td width="11"> </td>
<td width="181">
<div class="c2"><a href="http://news.nationalgeographic.com/news/2003/04/0410_030410_cannibal.html" target="_blank"><img src="../../BioFun-Support/AllGraphics/NoseBone.jpg" width="180" height="120" border="0" alt="** PLEASE DESCRIBE THIS IMAGE **"></a></div>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p class="c3">Since cannibalism is not nearly as common today, how does anyone get such diseases in the modern world?</p>
<p><span class="c5">There are rare cases of iatrogenic</span> transmission, that is, where the disease is caused by <a href="http://www.ncbi.nlm.nih.gov/pubmed/14522863" target="_blank">faulty medical practice</a>, for example through the use of contaminated surgical instruments or diseased tissue for transplantation.</p>
<p class="c3">Where did these people get the disease?  Since the disease is caused by the formation of <span class="c1">PrP<sup>sc</sup></span>, any event that leads to <span class="c1">PrP<sup>sc</sup></span> formation could cause the disease.</p>
<p><span class="c5">Normally, the formation of <span class="c1">PrP<sup>sc</sup></span> from <span class="c1">PrP<sup>c</sup></span> <span class="c8">is</span> very rare. We all have <span class="c1">PrP<sup>c</sup></span> but very few of us spontaneously develop kuru-like symptoms. </span><span class="c5">There are, however, mutations in the <span class="c1">gene that encodes PrP<sup>c</sup></span> that greatly enhance the frequency of the <span class="c1">PrP<sup>c</sup></span> → <span class="c8">PrPsc conversion.</span> </span></p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="390">
<p class="c3">Such mutations may be inherited (genetic) or may occur during the life of an organism (sporadic).</p>
<p class="style1 c9">The schematic of the brain on the left is from<br>
a person with <a href="http://www.ncbi.nlm.nih.gov/htbin-post/Omim/dispmim?600072" target="_blank">familial fatal insomnia (FFI)</a>.</p>
<p class="c3">FFI is due to the inheritance of a mutation in the <a href="http://www.ncbi.nlm.nih.gov:80/entrez/dispomim.cgi?id=176640" target="_blank">PRNP</a> gene; this mutation changes the normal aspartic acid 178 to an asparagine.</p>
<p class="c3">The squares mark brain regions affected; the lower section reveals the presence of aberrant protein aggregates in these regions.</p>
<p class="c3">When combined with a second mutation in the PRNP gene at position 129, the FFI mutation leads to <a href="http://www.ncbi.nlm.nih.gov/htbin-post/Omim/dispmim?123400" target="_blank">Creutzfeld-Jacob disease (CJD)</a>.</p>
</td>
<td width="10"> </td>
<td width="304">
<div class="c10"><img src="../../BioFun-Support/AllGraphics/FFI_CJD%20brains.jpg" width="296" height="344" alt="** PLEASE DESCRIBE THIS IMAGE **"></div>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<p><span class="c5">The circles in the right hand brain schematic indicate regions affected in CJD. There are holes or lacunae in the tissue. </span> <span class="c5">Both FFI and CJD are late onset diseases. Symptoms first appear ~ 50 years of age.</span></p>
</td>
</tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="19"></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="480">
<p class="c3">So why do <span class="c1">PrP<sup>sc</sup></span> aggregates accumulate?</p>
<p><span class="c5">To cut a peptide bond, a protease must position the target peptide bond within its catalytic active site.</span><span class="c5">If the target protein's peptide bonds do not fit into the active site, they cannot be cut.</span></p>
<p class="c3">Because of their structure, <span class="c1">PrP<sup>sc</sup></span> aggregates are highly resistant to proteolysis.</p>
<p class="c3">They gradually accumulate over many years, a fact that may explain the late onset of PrP-based diseases.</p>
</td>
<td width="10"> </td>
<td width="231">
<div class="c2"><a href="http://www.biochem.emory.edu/labs/genekdw/protdeg2000/why.html" target="_blank"><img src="../../BioFun-Support/AllGraphics/zincprotease.jpg" width="230" height="200" border="0" alt="** PLEASE DESCRIBE THIS IMAGE **"></a></div>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<div class="c2">for the pre-medical students along you.</div>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td><iframe width="720" height="540" src="http://www.youtube.com/embed/yzDQ8WgFB_U" frameborder="0"></iframe></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr noshade></td>
</tr>
</table>
<table class="c14" width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td class="c12" width="78">
<div class="c11">
<p class="white"><span class="white">Questions to answer</span></p>
</div>
</td>
<td class="c4" width="622"><span class="c5"> </span>
<ol>
<li>Why does the ribosome tunnel inhibit the folding of the newly synthesized polypeptide?</li>
<li>How would you expect protein degradation to be controlled?</li>
<li>How is the presence of a prion<sup>sc</sup> lead to the change in the structure of prion<sup>c</sup>, and is this necessarily a genetic change? </li>
<li>Why is it, do you think, that FFI and CJD are late onset diseases?</li>
<li>What, do you think, leads to the "lacunae" in brain tissue?</li>
</ol>
</td>
</tr>
<tr>
<td class="c13">
<div class="c11"><span class="whoe1 white">Questions to ponder</span><br></div>
</td>
<td class="c4">
<ul>
<li class="style21"><span class="style2">Why are proteins (and RNAs) degraded?</span></li>
<li class="style21"><span class="style2">Which do you think would be more susceptible to proteolytic degradation, a compact or an extended polypeptide?</span></li>
</ul>
</td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td>
<hr size="1"></td>
</tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td>replace with revised beSocratic activity</td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
<tr>
<td>
<hr size="1" noshade></td>
</tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
<td width="24"><a href="../../Biofundamentals-2012/lectureNotes-Revision/Topic3-6_Making Proteins.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="" width="24" height="24" border="0"></a></td>
<td width="29">
<div class="c16"><a href="#TOP"><span class="c15"><img src="../../BioFun-Support/AllGraphics/ArrowUP.GIF" width="24" height="24" alt="** PLEASE DESCRIBE THIS IMAGE **"></span></a></div>
</td>
<td width="637">
<div class="c17"><a href="https://twitter.com/share" class="twitter-share-button">Tweet</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>&nbsp; revised 
  <!-- #BeginDate format:En2 -->10-May-2014<!-- #EndDate --></div>
</td>
<td width="30">
<div class="c2"><a href="../../Biofundamentals-2012/lectureNotes-Revision/Topic3-8_Protein Activity.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowRIGHT.gif" width="24" height="24" border="0" alt="** PLEASE DESCRIBE THIS IMAGE **"></a></div>
</td>
</tr>
</table>
<script type='text/javascript'>
var _hl=_hl||{};_hl.site='2622';(function(){var hl=document.createElement('script');hl.type='text/javascript';hl.async=true;hl.src=document.location.protocol+'//highlighter.com/webscript/v1/js/highlighter.js';var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(hl,s);})();
</script>
</body>
</html>
