<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
   <meta name="Content-Type" content="text/html; charset=iso-8859-1">
   <meta name="description" content="Cell Biology & Development (with a little fun included).">
   <meta name="keywords" content="Klymkowsky, Boulder, Xenopus, intermediate filaments, cell adhesion, Wnt signaling, plakoglobin">
   <meta name="GENERATOR" content="Mozilla/4.75 (Macintosh; U; PPC) [Netscape]">
   <meta name="Description" content="Cell Biology  &  Development (with a little fun included).">
   <meta name="KeyWords" content="Klymkowsky, Boulder, Xenopus, intermediate filaments, cell adhesion, Wnt signaling, plakoglobin">
   <title>Biofundamentals -  PCR &amp; Subcloning </title>
<script language="JavaScript">
<!--
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
</head><style type="text/css">
<!--
a:link {text-decoration: none}
a:visited {text-decoration: none}
a:active {text-decoration: none}
a:hover {text-decoration: underline}
-->
</style>
<body text="#000000" bgcolor="#FCFDFF" link="#0000FF" vlink="#0066FF" alink="#0066FF" onLoad="MM_preloadImages('../../../Biofundamentals-2012/Biofundamentals/lectureNotes/graphics/week11graphics/EcoRIcut.gif')">
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="363" height="25"><font face="Arial, Helvetica, sans-serif"><font color="#006600"><b><a name="top"></a><font face="Arial, Helvetica, sans-serif" color="#006600">PCR-based 
      organism identification </font></b></font></font></td>
    <td width="175"> <div align="right"><font size="-2" face="Arial, Helvetica, sans-serif"><a href="javascript:window.close()" onMouseOver="window.status='Close this window';return true" onMouseOut="window.status='';return true">Close 
        this<br>
        window</a></font></div></td>
    <td width="54"><div align="right"><b><a href="javascript:window.close()" onMouseOver="window.status='Close Window';return true" onMouseOut="window.status='';return true"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/CommonGraphics/eyeblnk1.gif" alt="Close Window" width="53" height="31" border="0"></a></b></div></td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td> 
      <hr>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="381"> 
      <div align="left">
        <p><font face="Arial, Helvetica, sans-serif">For PCR analysis, we take 
          a sample and extract the nucleic acids present. </font></p>
        <p><font face="Arial, Helvetica, sans-serif">Because ribosomal RNAs are 
          so highly conserved, and yet display distinctive characteristics of 
          each species, whether bacterial, archaeal and eukaryal, it is common 
          to analyze ribosomal RNAs. </font></p>
        <p><font face="Arial, Helvetica, sans-serif">These RNAs are used to generate 
          complementary or <b><font color="#990000">cDNA</font></b> copies by 
          a process known as reverse transcription. </font></p>
        <p><font face="Arial, Helvetica, sans-serif">The enzyme that catalyzes 
          this process is, of course,<b><font color="#990000"> reverse transcriptase. 
          </font></b></font></p>
    <font face="Arial, Helvetica, sans-serif"><b> </b> </font></div>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td> 
      <p>&nbsp;</p>
      </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="368"><p><font face="Arial, Helvetica, sans-serif">It is used by 
        some viruses and other self-replicating, and inherently selfish, nucleic 
        acids to make DNA copies of RNA.</font></p>
      </td>
    <td width="20">&nbsp;</td>
    <td width="204"> 
      <p><a href="http://library.thinkquest.org/18258/retrovirus.htm" target="_blank"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/AllGraphics/dogma2.gif" width="344" height="94" border="0"></a></p>
      </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td> 
      <p><font face="Arial, Helvetica, sans-serif">PCR was made possible the discovery 
        of<a href="http://arbl.cvmbs.colostate.edu/hbooks/genetics/biotech/enzymes/hotpolys.html" target="_blank"><b> 
        thermostable DNA-dependent, DNA polymerases</b></a>. </font></p>
      <p align="right"><font face="Arial, Helvetica, sans-serif">Many such polymerases 
        have been isolated from <b><a href="http://www.bact.wisc.edu/Bact303/b27" target="_blank">thermophilic</a></b> 
        and <a href="http://www.sciam.com/0497issue/0497marrs.html#1" target="_blank"><b>hyperthermophilic</b></a> 
        archea. These organisms are known as <a href="http://www.micro.unsw.edu.au/rick/extremophiles.html" target="_blank"><b>extremophiles</b></a>!</font></p>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td> 
      <hr>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td> <p><font face="Arial, Helvetica, sans-serif">PCR is carried out using 
        a machine known as a <font color="#990000"><strong>thermocycler</strong></font>. 
        </font><font face="Arial, Helvetica, sans-serif">It heats and cools reactions 
        in an accurate and programmable manner. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">We set up reactions so that 
        they contains our target or <b><font color="#990000">template</font></b> 
        cDNA, two oligonucleotide <b><font color="#990000">primers</font></b>, 
        DNA <b><font color="#990000">polymerase</font></b>, and deoxynucleotide 
        triphosphates.</font></p></td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="154">
<p><font face="Arial, Helvetica, sans-serif">At the beginning of the reaction, 
        the temperature is increased. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">The double-stranded DNA template 
        denatures into single strands.</font></p>
      <p><font face="Arial, Helvetica, sans-serif">The primers are short, between 
        15 to 20 bases long. We designed these <font color="#990000"><strong>oligos</strong></font> 
        based on the sequences we wish t isolate. </font></p>
      </td>
    <td width="12">&nbsp;</td>
    <td width="429"> 
      <div align="right"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/AllGraphics/pcrTemp.jpg" width="428" height="233"></div>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td> 
      <hr>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="560"> 
      <p><font face="Arial, Helvetica, sans-serif">Primers are used in pairs - 
        one primer is directed against one strand in the DNA sequence to be amplified, 
        the other against the complementary strand. </font></p>
      </td>
    <td width="34" height="39"> <div align="center"><a href="http://www.dnalc.org/Shockwave/pcranwhole.html" target="_blank"><b><font face="Arial, Helvetica, sans-serif"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/CommonGraphics/important.jpg" width="23" height="23"><font size="-1"><br>
        PCR</font></font></b></a></div></td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td><p><font face="Arial, Helvetica, sans-serif">Their 3' OH ends point at 
        one another!</font></p>
      <p><font face="Arial, Helvetica, sans-serif">The primers are present at 
        many times the concentration of their target sequences within the original 
        <b><font color="#990000">template</font></b> DNA. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">As the temperature decreases, 
        the primers bind to, or <font color="#990000"><strong>anneal</strong></font>, 
        to complementary sequences in the template DNA. </font> </p></td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="368"> <img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/AllGraphics/pcrCartoon.jpg" width="368" height="255"></td>
    <td width="12">&nbsp;</td>
    <td width="212">
      <p><font face="Arial, Helvetica, sans-serif">The higher the annealing temperature, 
        the more specific the interaction. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Too high and no binding will 
        occur, too low and non-specific binding will predominate. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">After annealing, the temperature 
        is increased optimize the activity of the DNA polymerase, which extends 
        the primers, based on the template DNA sequence.</font></p>
      </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td> <font face="Arial, Helvetica, sans-serif">After a set amount of time, 
      the polymerase can synthesis ~1000 bases of DNA per minute, the temperature 
      is raised again to ~94 &deg;C to denature all of the strands in the reaction. 
      </font> </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="306"> 
      <p><font face="Arial, Helvetica, sans-serif">When the temperature is returned 
        to the annealing temperature, the cycle begins again. </font> </p>
      <p><font face="Arial, Helvetica, sans-serif">Each cycle leads to a doubling 
        of the number of target DNA. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">The resulting <b><font color="#990000">amplified 
        DNA</font></b> can be isolated from the reaction mix by a process known 
        as <font color="#990000"><strong>gel electrophoresis</strong></font>. 
        </font></p>
      <p><font face="Arial, Helvetica, sans-serif">To analyze individual amplified 
        DNA molecules, each originally copied from a specific template DNA molecule, 
        they must be be cloned. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">This is done by <font color="#990000"><strong>subcloning</strong></font> 
        pieces of DNA into a <font color="#990000"><strong>plasmid</strong></font>. 
        </font></p>
      </td>
    <td width="12">&nbsp;</td>
    <td width="274"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/AllGraphics/bluescript.jpg" width="273" height="280"></td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="276" height="86"> 
      <p align="left"><font face="Arial, Helvetica, sans-serif">In subcloning, 
        pieces of DNA are inserted into the plasmid. A plasmid is basically a 
        selfish piece of DNA. It <b>must</b> contain a DNA-replication origin 
        to be propagated in a bacterial cell. </font></p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">We then transform 
        the plasmid into a bacterial host. </font></p>
      </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td> 
      <hr>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="159" height="396"> 
      <p><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/AllGraphics/subcloning.jpg" width="156" height="377"></p>
      </td>
    <td width="12">&nbsp;</td>
    <td width="421"><font face="Arial, Helvetica, sans-serif">Most plasmids have 
      made themselves useful or nasty, some contain addition modules. </font> 
      <p><font face="Arial, Helvetica, sans-serif">One way to be useful is to 
        contain genes that encode antibiotic resistance. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">If a bacterium picks up a plasmid 
        that encodes an antibiotic resistance gene, that bacterium will become 
        resistant to the antibiotic. </font></p>
      <hr>
      <p><font face="Arial, Helvetica, sans-serif">To accept a DNA insert, the 
        plasmid must be cut -- generally using one or two <font color="#990000"><strong>restriction 
        endonucleases</strong></font>; these.are part of the host cell's defense 
        against foreign, selfish, DNAs, such as plasmids. </font></p>
      <table width="100%" border="0" cellpadding="0">
        <tr>
          <td width="55%"><div align="right"><font face="Arial, Helvetica, sans-serif">The 
              enzyme EcoRI cuts DNA at </font></div></td>
          <td width="45%"> 
            <div align="right"><a href="http://www.biochem.ucl.ac.uk/bsm/xtal/teach/rest/endo.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_swapImage('Image17','','../../../Biofundamentals-2012/Biofundamentals/lectureNotes/graphics/week11graphics/EcoRIcut.gif',1)" target="_blank"><img name="Image17" border="0" src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/AllGraphics/EcoRI.gif" width="220" height="34"></a></div>
          </td>
        </tr>
      </table>
      <p><font face="Arial, Helvetica, sans-serif">DNA molecules with compatible 
        ends can insert into this opening and be linked to the plasmid DNA by 
        a DNA ligase and ATP.</font> </p>
      </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td height="17"> 
      <hr>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td height="17"> 
      <p><font face="Arial, Helvetica, sans-serif">Generally the system is set 
        up so that plasmids are cut in such a way that if no DNA is inserted the 
        plasmid cannot be recircularized by the ligase. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Under most conditions, linear 
        DNA molecules are not replicated in bacteria. Circular DNA molecules, 
        containing an inserted DNA sequence, are replicated. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">While the process of insertion 
        and transformation is inefficient, we have many bacteria and a powerful 
        <font color="#990000"> <strong>selection</strong></font>. </font></p>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td height="17">&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td height="17"><p><font face="Arial, Helvetica, sans-serif">By plating the 
        transformed bacteria onto agar plates that contains the appropriate antibiotic, 
        only those bacteria that contain a circularized plasmid will grow. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">We <font color="#990000"><strong>select</strong></font> 
        for those bacteria that grow in the presence of the antibiotic, they must 
        contain stable plasmids. . </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Each bacteria with a plasmid 
        can grow as a clone into a visible colony. All of the cells of the clone 
        will contain copies of the same plasmid. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">We isolate the plasmid from 
        the bacteria and sequence the inserted DNA.</font></p>
      <p><font face="Arial, Helvetica, sans-serif">This DNA was originally derived, 
        via <font color="#990000"><strong>rtPCR</strong></font>, from a particular 
        RNA present in one of the organisms present in the original biofilm.</font></p>
      </td>
    <td height="17">&nbsp;</td>
    <td height="17"><a href="http://www.bact.wisc.edu/microtextbook/ClassAndPhylo/molPhylogeny.html" target="_blank"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/AllGraphics/rRNAtree.jpg" width="217" height="260" border="0"></a></td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td height="17">&nbsp; </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td height="17"> 
      <p><font face="Arial, Helvetica, sans-serif">E</font><font face="Arial, Helvetica, sans-serif">ach 
        type of organism contains its own characteristic ribosomal RNAs. Two distinct 
        species contain distinct rRNAs. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">The more the rRNAs of two organisms 
        differ, the further are the two organisms in evolutionary distance.</font></p>
      <p><font face="Arial, Helvetica, sans-serif">By searching through a <font color="#990000"><strong>library</strong></font> 
        of cloned sequences we can deduce the number of different organisms within 
        the original population! </font></p>
      <p><font face="Arial, Helvetica, sans-serif">From this information, we can 
        identify organisms and construct family trees. </font></p>
    </td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="510" bgcolor="#CCCCCC"> <hr /> </td>
    <td width="84" bgcolor="#CCCCCC"><div align="right"><a href="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/Questions/Block-3.htm#eight" target="_blank"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/CommonGraphics/questions.gif" width="84" height="15" border="0" /></a></div></td>
  </tr>
</table>
<table width="600" border="0" cellpadding="0">
  <tr> 
    <td width="363" height="25"><font face="Arial, Helvetica, sans-serif"><font color="#006600"><b><a href="#TOP"><font face="Arial, Helvetica, sans-serif" size="-2"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/CommonGraphics/ArrowUP.GIF" width="24" height="24"></font></a> 
      </b></font><font size="-2" face="Arial, Helvetica, sans-serif">Check the 
      <a href="http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?db=Books" target="_blank"><strong>NCBI 
      BookShelf</strong></a> | 3 December 2002</font><font color="#006600"><b> 
      </b></font></font></td>
    <td width="175"> <div align="right"><font size="-2" face="Arial, Helvetica, sans-serif"><a href="javascript:window.close()" onMouseOver="window.status='Close this window';return true" onMouseOut="window.status='';return true">Close 
        this<br>
        window</a></font></div></td>
    <td width="54"><div align="right"><b><a href="javascript:window.close()" onMouseOver="window.status='Close Window';return true" onMouseOut="window.status='';return true"><img src="../../../Biofundamentals-2012/Biofundamentals/lectureNotes/CommonGraphics/eyeblnk1.gif" alt="Close Window" width="53" height="31" border="0"></a></b></div></td>
  </tr>
</table>
<font face="Book Antiqua Italic"><b><font size=+1></font></b></font><font face="Book Antiqua Italic"></font><font face="Book Antiqua Italic"><b><font size=+1></font></b></font> 
</body>
</html>
