
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" 
	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
<meta name="Content-Type" content="text/html; charset=iso-8859-1" />
<meta name="Description" content="Interactive Introductory / Foundational Evolution and Molecular Biology" />
<meta name="Keywords" content="free on-line biology text, interactive molecular biology, interactive text, biology on-line" /><meta name="" content="free on-line biology, molecular biology, text" />
<meta name="Keywords" content="Biofundamentals, College-level, Foundational concepts, Klymkowsky, Evolution, Molecular Biology, Cell Biology, Interactive" />
   <meta name="GENERATOR" content="Mozilla/4.75 (Macintosh; U; PPC) [Netscape]" />

<title>Biofundamentals @ UC Boulder- Ancestors, analogies and homologies</title>
<script language="JavaScript" type="text/javascript">
<!--
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
<script src="../../../Biofundamentals/Scripts/AC_RunActiveContent.js" type="text/javascript"></script>
</head>
<head>
<style type="text/css">
<!--
a:link {text-decoration: none}
a:visited {text-decoration: none}
a:active {text-decoration: none}
a:hover {text-decoration: underline}

.style15 {font-family: Optima}
.style16 {font-size: small; font-family: Arial, Helvetica, sans-serif; }
.style17 {font-family: Arial, Helvetica, sans-serif}
.style18 {font-size: 12px; font-family: Optima; }
.style19 {font-family: Optima; font-size: small; }

.white {
	color: #FFF;
}
.white {
	color: #FFF;
}
.white1 {
	color: #FFF;
}
.style171 {font-family: Arial, Helvetica, sans-serif}
.style171 {font-family: Arial, Helvetica, sans-serif}
.style1711 {font-family: Arial, Helvetica, sans-serif}
.style1711 {font-family: Arial, Helvetica, sans-serif}
.style1711 {font-family: Arial, Helvetica, sans-serif}
.style1711 {font-family: Arial, Helvetica, sans-serif}
-->
</style>
</head>
<body text="#000000" bgcolor="#FCFDFF" link="#0000FF" vlink="#0066FF" alink="#0066FF" onload="MM_preloadImages('../../../Biofundamentals/BioFun-Support/AllGraphics/MegazostrodonBones.jpg','../../../Biofundamentals/Support/lectureNotes/AllGraphics/lineageB.jpg')">
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="643"><font face="Arial, Helvetica, sans-serif"><b>9. Recognizing
      relatives and ancestors: analogies &amp; homologies<font color="#006600" face="Arial, Helvetica, sans-serif"><a name="top" id="top2"></a></font><font face="Arial, Helvetica, sans-serif" size="-1"><br />
    </font></b></font></td>
    <td width="27">&nbsp;</td>
    <td width="25"><font face="Arial, Helvetica, sans-serif"><b><a href="Topic1H_Evo.htm" target="_self"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="arrow" width="24" height="24" border="0" /></a></b></font></td>
    <td width="25"><div align="right"><a href="Topic2AB_Water.htm" target="_self"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowRIGHT.gif" alt="arrow" width="24" height="24" border="0" /></a></div></td>
    <table width="700" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td><hr size="1" noshade="noshade" /></td>
    </tr>
</table>
  </tr>
</table>

<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr> 
    <td width="139" height="186"><img src="../../../Biofundamentals/BioFun-Support/wordles/page9.jpg" alt="wordle-1" width="320" height="180" align="middle" /></td>
    <td width="10">&nbsp;</td>
    <td width="551"><p><span class="style17">Well before the modern theory of evolution was proposed, it was obvious to naturalists, and particularly the Swedish naturalist <a href="http://www.ucmp.berkeley.edu/history/linnaeus.html" target="_blank">Carl Linneaus (1707-1778)</a>, that various organisms could be classified into a hierarchical scheme (you may have heard of it: kingdoms, phyla, orders, families, etc).</span></p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><div align="right">
      <p align="left"><span class="style17"><font face="Arial, Helvetica, sans-serif"><span class="style1711">In
                    this scheme, each type of organism is given a unique <span class="style4">place
                    and a </span></span><span class="style4">unique
        name.</span> The
                  most natural of these groups are &quot;species&quot;, which are defined
                  by the fact that organisms that </font></span><font face="Arial, Helvetica, sans-serif">are
                  of the same species can (and do) successfully interbreed with
                one another.&nbsp; </font></p>
<p align="left"><font face="Arial, Helvetica, sans-serif">The <a href="http://plato.stanford.edu/entries/species/" target="_blank">concept
    of a species</a> is complex, in large part because populations vary and
    evolve over time. The biological species concept was defined by Mayr.&nbsp;</font></p>
      <p align="left"><span class="style13"><em>&quot;Species are groups of actually (or potentially) interbreeding natural populations
        which are reproductively isolated from other such groups.&quot; - Ernst Mayr
        (1953)</em></span>    </p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">This definition
      is not completely unambiguous. For example, when,
      over the course of time does one species become distinct from its
      progenitor?&nbsp; How
      do you define a species when individuals do not interbreed.&nbsp; </font></p>
    </div></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="73%" class="style1"><p align="left"><font face="Arial, Helvetica, sans-serif">A
        particularly interesting example is known as a <span class="style4"><strong>ring
          species.</strong></span>&nbsp;Imagine populations
        of the species A; over the geographic range of A where there exist a number of subpopulations.&nbsp; </font></p>
<p align="left"><font face="Arial, Helvetica, sans-serif">These subpopulations
        (A<sub>1</sub> to A<sub>5</sub>)
        and (A<sub>a</sub> to A<sub>e</sub>) have limited regions of over-lap
        with one another, but where they overlap  they  interbreed successfully. </font></p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">But, populations
        A<sub>5</sub> and A<sub>e</sub> no longer interbreed successfully &ndash;
        are these populations separate species?
        &nbsp; </font></p></td>
    <td width="27%"><div align="right"><a href="http://www.actionbioscience.org/evolution/irwin.html" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ring species.gif" alt="ring" width="300" height="258" border="0" /></a></div></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="560"><p align="left" class="style1"><span class="style1"><font face="Arial, Helvetica, sans-serif"><strong>Defining
    relationships between species morphologically: </strong>The
        linnean classification system suggests a tree-like organization to
    the relationships between organisms.&nbsp; </font></span></p>
      <p align="left" class="style1"><span class="style1"><font face="Arial, Helvetica, sans-serif">How
        do we arrange organisms on such a </font></span><span class="style11"><font face="Arial, Helvetica, sans-serif">phylogenic</font></span><font face="Arial, Helvetica, sans-serif"><span class="style1"> tree? </span></font></p>
<p align="left"><span class="style1"><font face="Arial, Helvetica, sans-serif">One
        approach is known as <a href="http://www.fossilnews.com/1996/cladistics.html" target="_blank">cladistics</a>. Each type of organism is analyzed in terms of 
        primitive (ancestral) and derived (that is, more specialized)
        characteristics.&nbsp; </font></span></p>
      <p align="left"><span class="style1"><font face="Arial, Helvetica, sans-serif">Based
        on these traits, organisms can be arranged into <a href="http://www.amnh.org/exhibitions/Fossil_Halls/cladistics.html" target="_blank">trees. </a></font></span></p>
      <p align="left"><span class="style1"><font face="Arial, Helvetica, sans-serif">Of course,
        exactly how such a tree is structured depends upon which traits are
        considered to be primitive (and so present in ancestral forms) and
        which traits are considered to represent recent specializations<a href="http://www.amnh.org/exhibitions/Fossil_Halls/cladistics.html" target="_blank">.</a></font></span><font face="Arial, Helvetica, sans-serif"><br />
        </font></p></td>
    <td width="160"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/lineageA.jpg" width="160" height="160" alt="lineage A" /><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/lineageB.jpg" width="160" height="160" alt="lineage B" /></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><p class="style1"><font face="Arial, Helvetica, sans-serif">The ability to determine the complete sequence
                    of an organism's DNA has made it possible to use other traits,
                    DNA sequence and genome organization, to study
                    the relationship of organisms with respect to one another. </font></p>
    <p class="style1"><font face="Arial, Helvetica, sans-serif">In most cases this has confirmed anatomical based classification schemes, while in others it has revealed unexpected relationships.&nbsp; &nbsp;</font></p></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><hr size="1" noshade="noshade" /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td height="145"><p class="style1"><font face="Arial, Helvetica, sans-serif"><strong>Classification beyond species:&nbsp; </strong></font></p>
      <p class="style1"><font face="Arial, Helvetica, sans-serif">The next higher
              level of classification is &quot;genus&quot;.&nbsp; </font><font face="Arial, Helvetica, sans-serif">Organisms
              that are similar to one another, but  clearly distinct, can
              be placed in the same genus.&nbsp; For example there are a number
              of species in the genus  <em>Homo</em> [<a href="http://en.wikipedia.org/wiki/Human_evolution" target="_blank">link</a>],
              these include  <em> sapiens</em> (modern people), <em>floresiensis</em>, <em> habilis, </em> and <em> neanderthalensis</em>.&nbsp;</font></p>
    <p class="style1"><font face="Arial, Helvetica, sans-serif">Given their similarities, w</font><font face="Arial, Helvetica, sans-serif">e assume that all members of a particular genus share a common ancestor in the relatively recent (millions to tens of millions of years) past.&nbsp;</font></p></td>
    <td>&nbsp;</td>
    <td><div align="center"><a href="http://www.pandasthumb.org/pt-archives/000582.html" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/flor and sapiens.gif" width="184" height="141" border="0" /></a></div></td>
  </tr>
  <tr>
    <td height="184"><p class="style1"><font face="Arial, Helvetica, sans-serif">As
                we look deeper at the anatomical and molecular structure of organisms,
                particularly their genomic DNA sequences, we find evidence for
                deeper relationships.</font></p>
    <p class="style1"><font face="Arial, Helvetica, sans-serif">Humans, are grouped
            together with a number of living organisms (lemurs, lorises, tarsiers,
            new world monkeys, old world monkeys, and apes) and extinct species
            in the order of <a href="http://tolweb.org/treehouses/?treehouse_id=3029" target="_blank">Primates</a> [<a href="http://tolweb.org/Primates/15963" target="_blank">link</a>].&nbsp;</font></p>
    <p class="style1"><font face="Arial, Helvetica, sans-serif">The available
                    evidence indicates that all primates shared a common ancestor
                    that lived approximately 85 million years ago (that is about
    20 million years before the dinosaurs went extinct!).&nbsp;</font></p></td>
    <td>&nbsp;</td>
    <td><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/human-linnean-classification.jpg" width="200" height="186" alt="human classification" /></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
    <tr>
        <td><p class="style1"><font face="Arial, Helvetica, sans-serif">The primates
                    are divided into two suborders, all members of one of these,
                    the Haplorrhini, which includes tarsiers,
                    monkeys, apes, and human beings share
                    a  requirement for vitamin C in their diet.&nbsp;</font> <font face="Arial, Helvetica, sans-serif">This
                    requirement is due to a mutation in a gene (known
                    as GULO or 1-gulono-gamma-lactone oxidase). GULO1  is 
                    required for the synthesis of vitamin C.&nbsp; T</font><font face="Arial, Helvetica, sans-serif">his
      mutation in the ancestor or the Haplorrhini appears to have occurred ~40 million years ago. </font></p></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td width="282"><span class="style1"><font face="Arial, Helvetica, sans-serif"><a href="http://anthro.palomar.edu/earlyprimates/first_primates.htm" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/VitaminCPhylogeny.jpg" alt="vitamin C" width="281" height="183" border="0" /></a></font></span></td>
        <td width="418"><p class="style1"><font face="Arial, Helvetica, sans-serif">Putting </font><font face="Arial, Helvetica, sans-serif">
                    a working copy of this gene, <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=14962674&amp;query_hl=16&amp;itool=pubmed_docsum" target="_blank">derived
                    form a mouse</a>, into human
                                cells &quot;cures&quot; them
                  of the need for exogenous vitamin C. </font></p>
            <p class="style1"><font face="Arial, Helvetica, sans-serif">Guinea
                    pigs also require vitamin C, but since i) there are primates,
                    such as lemurs, that do not require vitamin C and ii) since
                    the 
                    common ancestor of primates and guinea pigs lived more
                   than 80 million years ago, it appears that the mutations that
                    make Haplorrhini  primates and guinea pigs vitamin C dependent
                    occurred independently. </font></p></td>
    </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
    <tr>
        <td><p class="style1"><font face="Arial, Helvetica, sans-serif"><span class="style171">Now the </span>deeper<span class="style171"> question is,  why should it be possible to group organisms into such a hierarchical system.&nbsp; Why should we share traits, at both the anatomical and DNA sequence level with lemurs and </span>tarsiers<span class="style171">?&nbsp; </span></font></p>
<p class="style1"><font face="Arial, Helvetica, sans-serif">Such an ordering
            of organisms is difficult to understand if each organism originated
            independently - if each were created de novo.</font></p>
          <p class="style1"><font face="Arial, Helvetica, sans-serif">On the other
            hand it makes complete sense if each modern organism evolved
            from an ancestor, and that that ancestor gave rise (over evolutionary time) to multiple species.&nbsp;The reason that organisms  appear related is that they <strong>are</strong>,
            they share a common ancestor.&nbsp;</font></p>
      <p class="style1"><font face="Arial, Helvetica, sans-serif">Here it is worth noting quite explicitly that while we are primates, no living primate is the ancestor to humans.&nbsp; What we share is a common ancestor that lived over 80 million years ago.&nbsp;</font></p></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td><hr size="1" /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td><p class="style1"><font face="Arial, Helvetica, sans-serif">Phylogenetic
                trees are generally compared based on the principle of <span class="style11">parsimony (</span><a href="http://en.wikipedia.org/wiki/Occam%27s_Razor" target="_blank">Occam's
            razor</a>.)&nbsp;  </font></p>
        <p class="style1"><font face="Arial, Helvetica, sans-serif">This principle
          states that, all else being equal, the simplest model that explains
          the data is preferable to a more complex one. When two
          models produce the same predictions and explain the same set of data,
      we  prefer the simpler of the two.</font></p>
        <p class="style1"><font face="Arial, Helvetica, sans-serif">In the case of the relationships between organisms, evolutionary mechanisms provide the simplest, and most predictive explanation.&nbsp;</font></p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0" bgcolor="#CCCCCC">
  <tr>
        <td width="80" bgcolor="#990000"><div align="center">
          <p><span class="white">Questions
          to answer</span>          </p>
          <p>&nbsp;</p>
          <p><br />
          </p>
        </div></td>
        <td width="640" bgcolor="#FFFFFF"> 
            <ol class="style17">
              <li><span class="style3">How would you go about determining whether
                an organism, identified through fossil evidence, was
                part of a new or a living species?&nbsp;&nbsp; </span></li>
              <li><span class="style3">How would you determine whether two
                species are part of the same genus? </span></li>
              <li><span class="style31">Why is the common need of primates
                for vitamin C evidence for   a common ancestor?</span></li>
              <li><span class="style31">Why do we assume that the mutations
                in Haplorrhini primates and guinea pigs were independent events?</span></li>
              <li>How could vitamin C dependence have arisen in the first place? </li>
              <li><span class="style31">What types of traits are likely to
                be primitive? </span></li>
              <li><span class="style31">How can you decide whether a trait
                is primitive or derived? Is it always unambiguous?</span></li>
            </ol></td>
    </tr>
    <tr>
        <td bgcolor="#0000FF"><div align="center"><span class="whoe1"><span class="white">Questions
                        to ponder</span></span><br />
        </div></td>
        <td bgcolor="#FFFFFF"><ul class="style17">
            <li><span class="style3">How does a species differ from a genus or
                    an order, are both equally &quot;natural&quot;
              groupings?&nbsp; </span></li>
            <li><span class="style3">What predictions does creation science/intelligent
              design make 
              about phylogenic relationships? </span></li>
        </ul></td>
    </tr>
</table>
<table width="720
" border="0" cellpadding="0" cellspacing="0">
  <tr>
        <td><hr size="1" /></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td class="style17"><strong>Homologs
            and Analogs </strong>
      <p align="left"><span class="style17">A major
            problem in classifying organisms is deciding whether similar traits
            were inherited from a common ancestor or whether they represent independent
            solutions to a particular problem. </span></p>
      <p align="left">For example, 
    an anatomical analysis of the forelimb of mammals<strong><font color="#990000"> </font></strong>suggests that they are <font color="#990000"><strong>homologous</strong></font> structures. </p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td class="style1"> <p align="left" class="style17">To say that two
           structures are homologous means that they are derived from a common
          ancestor and that 
        the feature  was present in that common ancestor. </p>
<p align="left" class="style17">This type of reasoning leads to the prediction
          that fossils of the last common ancestor of the mammals will be found
    to have a forelimb with a structure similar to that of modern mammals. </p></td>
    <td><a href="http://www.pbs.org/wgbh/evolution/library/04/2/l_042_01.html"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/TetrapodLimb.jpg" width="400" height="213" border="0" /></a></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td class="style1"> <p class="style17">Not all similar structures
          are homologous, however. The   alternative
          is that the  structures are <font color="#990000"><strong>analogous</strong></font>.
          When two structures are analogous, it means that they evolved
         independently, but perform the same functions or have a similar structure. </p>      </td>
  </tr>
</table>
<table width="700
" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td> <hr width="80%" size="1" /> </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="305"> <a href="http://www.nurseminerva.co.uk/adapt/wings.htm#bat" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/Wings.jpg" width="305" height="400" border="0" /></a></td>
    <td width="395" class="style1"><p align="left" class="style17">Their
                common form was determined by their common function. </p>
        <p align="left" class="style17">Consider 
          the wing of a pterodactyl, a flying reptile, a bird, and a bat, a mammal.</p>
      <p align="left" class="style17"> Analysis of
          these structures indicates that they are distinctly different from
              one another. </p>
      <p align="left" class="style17">In the pterodactyl,
          the wing membrane is supported by the 5th finger of the forelimb, in
          the bird by the 2nd finger, and in the bat, by the 3rd, 4th and 5th
          fingers.      </p>
      <p align="left" class="style17">The last common
          ancestor of flying reptiles,birds and mammals did not have wings,
          although it certainly had forelimbs.</p>
      <p align="left" class="style17">While wings of
          pterodactyls, birds, and bats are analogous 
      structures, their forelimbs are <strong class="style11">homologous</strong>. </p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td><p align="left" class="style1"><span class="style17">The physics of
          flight are constant. Organisms of similar size face the same aerodynamic
and thermodynamic         constraints. In general there are only a limited number
          of physically workable solutions to deal with these constraints.</span></p>
      <p align="left" class="style1"><span class="style17">Under
        these  conditions different populations will, through the process of
        variation  and selection, end up with structurally similar solutions.
        This process is known as 
        convergent evolution.&nbsp; </span></p>
      <p align="left" class="style1 style17"><font color="#000000">Convergent
              evolution</font> occurs when only certain solutions to
              a particular problem are evolutionarily accessible.&nbsp; <span class="style171">As another example, the
        use of a dagger is an effective solution to the problem of killing another
        organism.</span>.<br /> 
        </p>
      </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="446" height="207"><p align="left" class="style16">Variations of this
    solution have  been &quot;discovered&quot; or invented independently many
    times.</p>
      <p align="left" class="style16"><span class="style171">Marsupial 
      and placental mammals diverged from a common ancestor approximately 160Myr ago. </span></p>
<table width="100%" border="0" cellpadding="0" cellspacing="0">
        <tr>
          <td width="33%"><a href="http://www.indiana.edu/%7Eensiweb/c.sinoc.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Image13','','../../../Biofundamentals/BioFun-Support/AllGraphics/MegazostrodonBones.jpg',1)"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/MegazostrodonFlesh.jpg" name="Image13" width="140" height="116" border="0" id="Image13" /></a></td>
          <td width="67%" class="style1"><p align="left" class="style17">Since 
          then, they have converged on a number of similar adaptations.</p></td>
        </tr>
      </table>    </td>
    <td width="10"> 
      <p>&nbsp;</p>    </td>
    <td width="264"><p>
       
    <iframe width="425" height="240" src="http://www.youtube.com/embed/ntVI4N7AUx0?rel=0" frameborder="0" allowfullscreen></iframe></p>    </td>
  </tr>
</table>
<table width="700
" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td><hr size="1" /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p align="left" class="style16"><strong>The
            loss of traits </strong></p>
        <p align="left" class="style17"><span class="style1">Another
              process that can confuse or complicate cladistic analysis is the
              loss of traits.&nbsp; </span></p>
        <p align="left" class="style17"><span class="style1">As
                organisms adapt to a specific environment and life style, traits
                once useful can become irrelevant and may be lost. </span></p>
        <p align="left"><span class="style16">A
              classic example is the reduction of  hind limbs during the <a href="http://ngm.nationalgeographic.com/ngm/data/2001/11/01/html/ft_20011101.4.html?fs=animals-panther.nationalgeographic.com" target="_blank">evolution
              of whales</a>.&nbsp; </span><span class="style1"><br />
        </span></p>        </td>
    <td>&nbsp;</td>
    <td><a href="http://www.pbs.org/wgbh/evolution/library/03/4/l_034_05.html" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/whale hindlimb.gif" width="401" height="208" /></a></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td><a href="http://www.aloha.net/%7Esmgon/eyes.htm" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/eyeless trilobites.jpg" width="83" height="130" border="0" /></a></td>
    <td>&nbsp;</td>
    <td><p><span class="style17">Another
      common loss is that of eyes, often seen as populations adapt
      to environments in which light is absent.&nbsp;The
      loss of traits can itself be an adaptation if it provides an advantage
      to organisms living in a particular environment.&nbsp;       </span></p>
      <p><span class="style17">This
        fact can make it difficult to determine whether an organism is primitive
      (that is retains ancestral features) or is highly evolved.</span><span class="style16 style17">&nbsp; </span></p></td>
  </tr>
</table>


<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="400" class="style1"> <p align="left" class="style17"><strong>Evidence
              for a common ancestor: </strong> Similar structures
                  may arise independently, while apparently different structures,
          such as a human hand and a whale's flipper, are  related. </p>
<p align="left" class="style17">How do we distinguish
           between these two possibilities?&nbsp; In general, we carry out a
           detailed comparative analysis of the development and final form of
           the structure.&nbsp; </p>
      <p align="left" class="style17">In the modern
          world of molecular biology, this involves the genes that are  active
          as the structure forms.&nbsp; </p>
      <p align="left" class="style17">The more details
          two structures share, the more likely they are to be homologous.</p></td>
  </tr>
</table>
<a href="http://i.imgur.com/emPJ3.gif" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/phylogenetic-tree-and-time.gif" width="720" height="290" alt="phylo" /></a>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td class="style1"><p class="style17">Detailed
          analysis of many different types of organisms reveals the presence
          of a common molecular signature that
          strongly suggests that all living organisms are closely related; that
          is, that they  share a common ancestor. </p>
      <p class="style17">What are these apparent universal homologies?
          They range from the basic  structure of cells to the
         molecular machinery involved in energy capture and transduction, information
         storage and utilization.&nbsp; </p>
      <p class="style17">To review (superficially),  all extant
          organisms  ...</p>
      <ul>
          <li class="style17"> use double stranded DNA as
            their genetic material</li>
          <li class="style17">use the same molecular
                systems, <span class="style11">transcription</span> and <span class="style11">translation</span>, to
                access the information stored in DNA              </li>
          <li class="style17">use a common genetic code<strong><font color="#990000">, </font></strong>with few variations, to specify the sequence
            of polypeptides (proteins)</li>
          <li class="style17">use ribosomes to translate the information stored in messenger RNAs into polypeptides</li>
          <li class="style17"> share common enzymatic
                  (metabolic) pathways </li>
      </ul></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td><hr size="1" noshade="noshade" /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0" bgcolor="#CCCCCC">
  <tr>
    <td width="80" bgcolor="#990000"><div align="center">
      <p><span class="white1">Questions
        to <span class="white">answer</span></span></p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p><br />
      </p>
    </div></td>
    <td width="640" bgcolor="#FFFFFF"><ol>
      <li class="style17">What typical mammalian traits have whales
        lost during their evolution? which have they retained?</li>
      <li class="style17">How does the loss of a trait complicate
        a cladistic analysis?&nbsp;</li>
      <li class="style17">Does the premise of &quot;intelligent design&quot; make
        any clear or testable predictions about the apparent relatedness
        of different species? Is it useful for understanding homologous
        and analogous structures? </li>
      <li class="style17">Is convergent evolution possible
        at the molecular level?&nbsp;&nbsp; </li>
      <li class="style17">If all organisms are
        descended from a common ancestor, what can we say about the
        diversity of pre-biogenic systems that existed before that
        ancestor? </li>
    </ol></td>
  </tr>
  <tr>
    <td bgcolor="#0000FF"><div align="center"><span class="whoe1"><span class="white1">Questions
      to ponder</span></span><br />
    </div></td>
    <td bgcolor="#FFFFFF"><ul>
      <li class="style17">How does the presence of convergent
        evolution complicate phylogenic analyses?&nbsp; What general types
        of conditions might lead to a complex organism becoming
        simpler?</li>
      <li class="style17">What benefit(s) could be linked to the loss of
        eyesight or other
        &quot;advanced&quot; traits?&nbsp;</li>
    </ul></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" /></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td width="131"><table width="100%" border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td width="58"><a href="https://solarsystem.colorado.edu/conceptInventories/external/bioliteracy.php" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/eyeblnk1.gif" alt="eye" width="53" height="31" border="0" /></a></td>
        <td width="662"><div align="left" class="style17"><strong>before class</strong>&nbsp; [ talk amongst yourselves about the questions]</div></td>
      </tr>
    </table>
      <table width="720" border="0" cellspacing="0" cellpadding="0">
        <tr>
          <td><hr size="1" noshade="noshade" /></td>
        </tr>
        <tr>
          <td width="580"><iframe width="720" height="574" frameborder="0" src="http://besocratic.clemson.edu/besocratic.html?Activity=751#/ActivityViewerPage"></iframe></td>
        </tr>
      </table></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
        <td><hr size="1" noshade="noshade" /></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
      <td width="25"><font face="Arial, Helvetica, sans-serif"><b><a href="Topic1H_Evo.htm" target="_self"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="arrow" width="24" height="24" border="0" /></a></b></font></td>
    <td width="29"><div align="left"><a href="#TOP"><font size="-2"> <img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowUP.GIF" width="24" height="24" /></font></a></div></td>
    <td width="635"><div align="right"><font size="-2">revised
    <!-- #BeginDate format:En2 -->31-Aug-2012<!-- #EndDate -->
    </font></div></td>
    <td width="10">&nbsp;</td>
    <td width="24"><a href="Topic2AB_Water.htm" target="_self"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowRIGHT.gif" alt="arrow" width="24" height="24" border="0" /></a></td>
  </tr>
</table>
<script type='text/javascript'>var _hl=_hl||{};_hl.site='41';(function(){var hl=document.createElement('script');hl.type='text/javascript';hl.async=true;hl.src=document.location.protocol+'//highlighter.com/webscript/v1/js/highlighter.js';var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(hl,s);})();</script> 

</body>
</html>

