<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
   <meta name="Content-Type" content="text/html; charset=iso-8859-1" />
   <meta name="description" content="Interactive Introductory / Foundational Evolution and Molecular Biology" />
   <meta name="keywords" content="free on-line biology text, interactive molecular biology, interactive text, biology on-line" /><meta name="" content="ffree on-line biology, molecular biology, text" />
   <meta name="keywords" content="Biofundamentals, College-level, Foundational concepts, Klymkowsky, Evolution, Molecular Biology, Cell Biology, Interactive" />
   <meta name="GENERATOR" content="Mozilla/4.75 (Macintosh; U; PPC) [Netscape]" />
   
   
   <title>Biofundamentals @ UC Boulder - Peptide bonds and polypeptides</title>
<script language="JavaScript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
//-->
</script>
<style type="text/css">
<!--
.style1 {color: #000000}
-->
</style>
<script src="../../../Biofundamentals/Scripts/AC_RunActiveContent.js" type="text/javascript"></script>
</head><style type="text/css">
<!--
a:link {text-decoration: none}
a:visited {text-decoration: none}
a:active {text-decoration: none}
a:hover {text-decoration: underline}
body,td,th {
	font-family: Arial, Helvetica, sans-serif;
}
.style2 {
	color: #990000;
	font-weight: bold;
}
.style4 {font-size: x-small}
.white {
	color: #FFF;
}
.cayenne {
	color: #800000;
}
-->
</style>
<body bgcolor="#FCFDFF" text="#000000" link="#0000FF" vlink="#0066FF" alink="#0066FF" onLoad="MM_preloadImages('../../../Biofundamentals/Support/lectureNotes/AllGraphics/DLMovie.gif','../../../Biofundamentals/Support/lectureNotes/AllGraphics/PrPc acidic.jpg','../../../Biofundamentals/BioFun-Support/AllGraphics/PrPc acidic.jpg')">
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="512"><div align="left"><font color="#000000" face="Arial, Helvetica, sans-serif"><b>22. Peptide
      bonds, polypeptides and proteins </b></font><font face="Arial, Helvetica, sans-serif" size="-1"><br />
    </font></div></td>
    <td width="47"><font color="#006600" face="Arial, Helvetica, sans-serif"><b><a name="top"></a></b></font></td>
    <td width="112">&nbsp;</td>
    <td width="24"><a href="Topic3-4_Mutation.htm" target="_self"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="" width="24" height="24" border="0" /></a></td>
    <td width="25"><div align="right"><a href="Topic3-6_Making Proteins.htm" target="_self"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowRIGHT.gif" alt="arrow" width="24" height="24" border="0" /></a></div></td>
    <table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" noshade /></td>
  </tr>
</table>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
<tr>
        <td>&nbsp;</td>
  </tr>
</table>


<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif">We have already mentioned proteins a number of times,
          since there are few   biological processes that do not rely
          on them. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Proteins derive their name
          from the ancient Greek sea-god <a href="http://www.olympian-foundation.org/proteus.htm" target="_blank">Proteus </a> who,
          like your typical sea-god, could change shape. The name acknowledges
    the many different properties and functions of proteins.&nbsp;&nbsp;</font></p></td>
    <td>&nbsp;</td>
    <td><img src="../../../Biofundamentals/BioFun-Support/wordles/page18.jpg" width="330" height="140" alt="wordle-1" /></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif">Proteins
          can act as catalysts (enzymes) and regulators of chemical reactions &ndash; we
    have already seen how proteins act to regulate transport across membranes.&nbsp;Proteins
      control the expression of genes,
      how genes respond to internal and external signals, and the replication
      of the genetic material.</font></p>
      <p><font face="Arial, Helvetica, sans-serif">Proteins</font><font face="Arial, Helvetica, sans-serif"> can act as structural
        components, determining both the shape and mechanical properties of
        cells and tissues.&nbsp; They can be motors, responsible
        for movements <strong>within</strong> cells and the movement of cells,
    tissues and the organism as a whole.&nbsp; </font></p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="182"> <p><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/AminoAcid.jpg" alt="" width="182" height="120" /></p></td>
    <td width="10"><p>&nbsp;</p></td>
    <td width="508"><p><font face="Arial, Helvetica, sans-serif">Proteins are 
      composed of  &alpha;-amino 
        acids linked together by peptide bonds into polypeptide chains. </font></p>
      <p>We can think of an amino acid as a modified form of methane (CH<font face="Arial, Helvetica, sans-serif"><strong><sub>4</sub></strong></font>).&nbsp; Instead
          of four hydrogens attached to the&nbsp;central C, there is one H, an<font face="Arial, Helvetica, sans-serif"> amino <font color="#000000">group</font> (<strong>-NH<sub>2</sub></strong>),
          a carboxylic  acid group (<strong>-COOH</strong>), and a final, variable
          (R) group attached to the central &quot;&alpha;-carbon&quot;. </font></p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="503"> <p><font face="Arial, Helvetica, sans-serif">The various R-groups are often referred to as &quot;side-chains&quot;. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">The four groups attached to 
        the </font> <font face="Arial, Helvetica, sans-serif"> <span class="style1">&alpha;</span>-carbon 
          are arranged at the vertices of a tetrahedron. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">If all four groups attached
            to the <span class="style1">&alpha;</span>-carbon
            are different from one another, as they are in all amino acids except
            glycine, the amino acid can exist in two possible stereoisomers,
      which are known as enantiomers.</font></p></td>
    <td width="13">&nbsp;</td>
    <td width="184"><a href="http://mathworld.wolfram.com/Tetrahedron.html" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/tetrahedron.jpg" alt="" width="180" height="180" /></a></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="248"><a href="http://biomedia.bio.purdue.edu/IML/Amino/html/stereoisomers.html" target="_blank" onMouseOver="MM_swapImage('Image12','','../../../Biofundamentals/BioFun-Support/AllGraphics/DLMovie.gif',1)" onMouseOut="MM_swapImgRestore()"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/DLAminoAcids.jpg" name="Image12" width="248" height="128" border="0" /></a> 
    </td>
    <td width="10">&nbsp;</td>
    <td width="442"><p><font face="Arial, Helvetica, sans-serif">Enantiomers
           are mirror images of one another and are termed the L- and D- forms. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Only
           L-type amino acids are found in proteins, even though there is <a href="http://www.bio.puc.cl/pap03.htm" target="_blank">no
            obvious  reason</a> that proteins could not have also been
            made using both types of amino acids, or using D-amino acids.</font></p>
    </td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif">It is not that D-amino acids
          do not occur in nature, or in organisms, they do.&nbsp;</font><font face="Arial, Helvetica, sans-serif">They
            are found in biomolecules, such as the antibiotic gramicidin, which
            is composed of alternating L-and D-type amino acids &ndash; however
            gramicidin is synthesized by <a href="http://csbcc1.csbcc.dartmouth.edu/nrps.html" target="_blank">a
            different process</a> than that used to synthesize
            proteins.  </font></p>
      <p><font face="Arial, Helvetica, sans-serif">It appears that
          the universal use of L-type amino acids is yet another example of the
      evolutionary relatedness of organisms (it is a homologous trait). </font> <font face="Arial, Helvetica, sans-serif">Even though there 
            are hundreds of different amino acids known, only 19 amino acids
          and one imino acid, proline,
      are found in proteins.</font></p>
    </td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td>&nbsp; </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="280"> 
<iframe width="400" height="300" src="http://www.youtube.com/embed/L9L6clXbJD8?rel=0" frameborder="0" allowfullscreen></iframe>
<br></td>
    <td width="10">&nbsp; </td>
    <td width="430"><p><font face="Arial, Helvetica, sans-serif">These amino acids differ in 
        their R-groups. </font></p>
<p><font face="Arial, Helvetica, sans-serif">Some of these R-groups are 
        highly hydrophobic, some are hydrophilic, and some of the hydrophilic R-groups are positively or negatively 
        charged at physiological pH. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">The different R-groups provide 
      proteins with a range of chemical properties.</font></p>
      <p>[<a href="../../../Biofundamentals/BioFun-Support/AllGraphics/AminoAcidViewer.swf" target="_blank">link
      to amino acid viewer</a>]</p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr> </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" noshade /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td><p><font face="Arial, Helvetica, sans-serif"><strong>Building a polypeptide:&nbsp; </strong></font></p>
      <p><font face="Arial, Helvetica, sans-serif">As we noted for nucleic acids,
              a polymer is a chain of subunits, monomers, linked together by
              chemical bonds.&nbsp;The bond between two amino
               acids is known as a peptide bond<font color="#000000">.
        Two amino acids, joined together by a peptide bond, is known as a dipeptide<font color="#990000"><strong>.</strong></font></font></font></p>
<p><font face="Arial, Helvetica, sans-serif"><font color="#000000">An
                      amino acid polymer is known as a polypeptide</font>. 
                 A polypeptide can be composed of 20 different possible
                monomers that can, in theory, be linked together in any imaginable
    order. </font>      </p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td width="490"><p><font face="Arial, Helvetica, sans-serif">The formation of a peptide bond involves what is known as a condensation
      or sometimes a &quot;dehydration&quot; reaction.&nbsp; In a condensation
                            reaction</font> two molecules
                      are joined together and a molecule of water is released. </p>
            <p><font face="Arial, Helvetica, sans-serif">In a hydrolysis
                                reaction</font>, the reverse occurs. <font face="Arial, Helvetica, sans-serif">The addition of a water
                    molecule is associated with splitting the original molecule
        into two parts. </font></p></td>
        <td width="104"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/peptideBond.gif" width="348" height="213" /></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
        <td><p>In the case of both nucleic acids and polypeptides, polymer assembly
            involves a condensation reaction (with the release of water), while
        polymer disassembly into monomers involves a hydrolysis reaction (with water as
            a reactant). </p>
          <p><font face="Arial, Helvetica, sans-serif">Combining amino acids
                  into a polypeptide</font><font face="Arial, Helvetica, sans-serif"> can
                  produce a very large number of possible polypeptides. There
            are 20<sup>100</sup> possible different
      polypeptides that are 100 amino acid residues long.&nbsp;</font>Many biological
      polypeptides are much longer!</p>
      <p><font face="Arial, Helvetica, sans-serif">During the
                    synthesis of a polypeptide, an amino acid is attached to
                    the C- or carboxyl terminus of the existing chain.</font><font face="Arial, Helvetica, sans-serif"> This
      generates an unbranched, linear polymer. </font></p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
<tr>
        <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td width="480">

          <iframe width="480" height="360" src="http://www.youtube.com/embed/_5kbJlqInFs?rel=0" frameborder="0" allowfullscreen></iframe>

</td>
        <td width="18">&nbsp;</td>
        <td width="222"><p><font face="Arial, Helvetica, sans-serif">The peptide bond has a number
  of characteristics that are critical in determining how polypeptides
  behave. 
  </font></p>
          <p><font face="Arial, Helvetica, sans-serif">Although drawn as a single bond, the peptide bond behaves more
            like a double  bond.</font></p>
<p><font face="Arial, Helvetica, sans-serif">While there
                is free rotation around a single bond, rotation around a peptide
          bond is constrained. </font></p>
<p align="left"><font face="Arial, Helvetica, sans-serif">Proline peptide
  bonds are found in the cis</font> configuration ~100 times as
  often as those between other amino acids.</p></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="140"><p><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/prolineKink.jpg" alt="" width="140" height="130" /></p></td>
    <td width="10">&nbsp;</td>
    <td width="570"><p><font face="Arial, Helvetica, sans-serif">The carbonyl 
  oxygen</font> (-C=O) acts as a H-bond acceptor, 
  while the amino hydrogen 
  (-N-H) acts as a H-bond donor. </p>
      <p><font face="Arial, Helvetica, sans-serif">There is <strong>no
        H-bond donor</strong> in the proline peptide bond </font><font face="Arial, Helvetica, sans-serif">and
          the presence of proline leads to a bend or kink in the
        polypeptide chain.</font></p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
<tr>
        <td>&nbsp;</td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="485"><iframe width="480" height="360
   
   " src="http://www.youtube.com/embed/_xF96sNWnK4?rel=0" frameborder="0" allowfullscreen></iframe></td>
    <td width="13">&nbsp;</td>
    <td width="222"><p>In this molecular level simulation, you cannot see the water, but you can watch this single polypeptide chain fold; the spinning of side chains around single bonds is very clear. </p>
    <p>The colors indicate where an animo acid is along the polypeptide chain.&nbsp;&nbsp;</p>
    <p>Notice
        that molecular movement continue even after the molecule reaches its
        compact (presumably native) state. </p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td><hr size="1" noshade /></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0" bgcolor="#CCCCCC">
  <tr>
    <td width="80" bgcolor="#990000"><div align="center" class="white">
      <p>Questions to answer      </p>
      <p><br />
      </p>
    </div></td>
    <td width="640" bgcolor="#FFFFFF"><font face="Arial, Helvetica, sans-serif">&nbsp; </font>
      <ol>
        <li> What limits  the structure of an R-group?&nbsp;</li>
        <li>Are there any theoretical constraints on the order of monomers in
          a polypeptide?</li>
        <li><span class="style5">Consider a condensation reaction; how would
          the reaction be effected if we removed water from the system?&nbsp;&nbsp;</span></li>
        <li>Why does it matter that rotation around a peptide bond is &quot;constrained&quot;?</li>
      </ol></td>
  </tr>
  <tr>
    <td bgcolor="#0000FF"><div align="center"><span class="whoe1"><span class="white">Questions to ponder</span></span><br />
    </div></td>
    <td bgcolor="#FFFFFF"><ul>
      <li class="style21">Is the use of a common set of amino acids among living organisms
        an analogous or a homologous trait? </li>
      <li class="style21">What factors can you imagine influenced the &quot;universal&quot; set of amino acids used in organisms?</li>
    </ul></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" noshade /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td> <p><font face="Arial, Helvetica, sans-serif"><strong>Water, polypeptide
            synthesis and folding:&nbsp; </strong></font></p>
      <p><font face="Arial, Helvetica, sans-serif">Were it not 
        for the presence of hydrophobic R-groups, all polypeptides would assume
        an extended configuration in water.&nbsp;&nbsp;H-bond
        donors and acceptor  groups within the polypeptide backbone would form H-bonds
        with each other and, because there are so many of them, with water molecules. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">Typical polypeptides
        have all of the different R-groups in various proportions. We write the
        amino acid sequence of a polypeptide from its N- or amino
          terminus</font> to its C- or carboxyl
          terminus, with the N-terminus to the left and the C-terminus
      to the right.&nbsp;<font face="Arial, Helvetica, sans-serif">The sequence
        of amino acid residues, that is what is left of the amino acid once
        it is part of the polypeptide, along the length of a polypeptide from its N- to its C-terminus, is known as its primary
      structure</font>.</p>
<p><font face="Arial, Helvetica, sans-serif">A number of
          the amino acid R-groups are  <a href="http://www.russell.embl-heidelberg.de/aas/hydrophobic.html" target="_blank">hydrophobic</a>. Their presence makes an extended configuration
      for the polypeptide energetically unfavorable. </font></p>
<p><font face="Arial, Helvetica, sans-serif">Very much like the process
               that drives the assembly of lipids into micelles and bilayers, a
              typical polypeptide  in aqueous solution will collapse onto itself
              in order to remove as many of its hydrophobic
              residues</font> 
      from contact with water as possible (otherwise,  water molecules would be organized around the hydrophobic residues, leading to a decrease in entropy!)<br />
<br />
</p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="513"> <p><font face="Arial, Helvetica, sans-serif">It is a basic assumption of
          structural biology that the final folded state of a polypeptide is
          determined by the sequence of amino acids along its length, and that
    this final structure is the state of lowest, or close to the lowest, energ</font>y. </p>
      <p><font face="Arial, Helvetica, sans-serif">Protein structure
        is commonly presented in a hierarchical</font> 
        manner. <font face="Arial, Helvetica, sans-serif">While this is an over-simplification, 
    it is a good place to start.</font></p></td>
    <td width="10" bgcolor="#FFFFFF">&nbsp;</td>
    <td width="200">
<iframe width="400" height="300" src="http://www.youtube.com/embed/Rq7DwrX0Uoc?rel=0" frameborder="0" allowfullscreen></iframe>
</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p align="left">&nbsp;</p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p align="left"><font face="Arial, Helvetica, sans-serif">As it is being synthesized,
          the process known as translation</font>,
          the polypeptide chain begins to fold.&nbsp;<font face="Arial, Helvetica, sans-serif">We
    can think of the folding process as a walk across an energy landscape.&nbsp;</font></p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">The goal is 
          to find the lowest point in the landscape, the energy minimum of the 
    system. </font><font face="Arial, Helvetica, sans-serif">This is generally assumed to be the native</font> or functional state of the polypeptide.</p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p align="left">&nbsp;</p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="240"><p><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/FoldingLandscape.jpg" width="240" height="227" /> 
      </p>
    </td>
    <td width="15">&nbsp;</td>
    <td width="445"><div align="right">
      <p align="left"><font face="Arial, Helvetica, sans-serif">The first step is the movement
        of hydrophobic R-groups out of contact with water.&nbsp;</font><font face="Arial, Helvetica, sans-serif">This
          drives the collapse of the polypeptide into a compact and dynamic &quot;molten
        globule</font>&quot;. </p>
<p align="left"><font face="Arial, Helvetica, sans-serif">The path to
  the native state is not necessarily a smooth or predetermined one. </font></p>
<p align="left"><font face="Arial, Helvetica, sans-serif">The folding polypeptide can
           get &quot;stuck&quot; in a local energy minimum; there may not be
        enough energy (derived from thermal collisions) for it to get out again.&nbsp; </font></p>
    </div></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="469"><p><font face="Arial, Helvetica, sans-serif">If a polypeptide gets stuck,
           there are mechanisms to unfold it and let it try again to reach
          its  native state. This
    process of partial unfolding is carried out by proteins known as chaperones</font>. </p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
<tr>
        <td><p align="left">&nbsp;</p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
        <td width="281"><p align="left"><a href="http://www.pnas.org/cgi/content/abstract/98/6/2985" target="_blank" onMouseOver="MM_swapImage('Image13','','../../../Biofundamentals/BioFun-Support/AllGraphics/PrPc acidic.jpg',1)" onMouseOut="MM_swapImgRestore()"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/PrPc neutral.jpg" alt="protein" name="Image13" width="281" height="287" border="0" /></a></p></td>
        <td width="10">&nbsp;</td>
        <td width="429"><p align="left"><font face="Arial, Helvetica, sans-serif">Some
                    amino acid R-groups contain carboxylic acid or amino groups,
                    and so also act as weak acids and bases. </font></p>
            <p align="left"><font face="Arial, Helvetica, sans-serif">Depending
                    on the pH of the solution they are in, these groups may be
                    protonated or unprotonated (that is, charged). </font></p>
            <p align="left"><font face="Arial, Helvetica, sans-serif">Whether
                    a group is charged or uncharged can have a dramatic effect
                    on the structure, and therefore the activity, of a protein. </font></p>
            <p align="left"><font face="Arial, Helvetica, sans-serif">By regulating
                    pH, an organism can modulate the activity of specific proteins. There
                    are, in fact, compartments within eukaryotic cells that are
        maintained at low pH in part to regulate protein structure and protein
                    activity.</font></p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" noshade /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td><p><font face="Arial, Helvetica, sans-serif"><strong>Secondary structure
            features:&nbsp; </strong></font></p>
      <p><font face="Arial, Helvetica, sans-serif">All polypeptides share a common 
        backbone structure made up of peptide bonds. It 
        is therefore not surprising that there are common patterns in
        polypeptide folding.</font></p>
        <p><font face="Arial, Helvetica, sans-serif">The first of these to be recognized, the  
        
&alpha;-heIix, was discovered by <a href="http://www.paulingexhibit.org/exhibit/alpha-helix.html" target="_blank">Linus
 Pauling</a> and Robert Corey, and reported in 1951.</font></p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">This was followed
           shortly thereafter by their  description of the &beta;-sheet. </font></p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">The forces that
          drive the formation of the
&alpha;-helix and the &beta;-sheet will be familiar. They are they same forces
 that underlie water structure.</font></p>
      <p align="right"><font size="-2" face="Arial, Helvetica, sans-serif">Linus
        at age 5</font></p></td>
    <td>&nbsp;</td>
    <td><a href="http://www.orst.edu/dept/lpi/lpbio/lpbio2.html" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/youngLinus.jpg" width="103" height="280" border="0" /></a></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td width="119"> <p><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/alphaHelix.jpg" width="113" height="200" /><font face="Arial, Helvetica, sans-serif"> 
        </font></p></td>
    <td width="11">&nbsp;</td>
    <td width="389"> <p><font face="Arial, Helvetica, sans-serif">In an </font><font face="Arial, Helvetica, sans-serif"> &alpha;-helix
                and a &beta;-sheet, all of the possible H-bonds  involving the
                peptide bond's donor and acceptor groups are formed within the polypeptide. </font></p>
        <p><font face="Arial, Helvetica, sans-serif">This
            stabilizes the structure. </font></p>
      <p><font face="Arial, Helvetica, sans-serif">In an &alpha;-helix , 
    the R-groups point outward from the long axis of the helix. </font></p></td>
    <td width="181"><div align="right"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/axialhelix.jpg" width="180" height="191" /></div></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td>&nbsp; 
    </td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr> 
    <td> <p><font face="Arial, Helvetica, sans-serif">In &beta;-sheets, the H-bonds 
        between the peptide bond donor and acceptor groups are made between either 
        parallel </font>or anti-parallel polypeptide chains. </p>
      <p align="right"><font size="-2" face="Arial, Helvetica, sans-serif">These 
        are anti-parallel &szlig;-sheets</font></p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">The R-groups point 
        out of and into the plane of the sheet. </font></p>
    </td>
    <td>&nbsp;</td>
    <td><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/betasheet.gif" width="429" height="256" /></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif">A key factor that determines
          the specific details of polypeptide folding are the interactions between
    R-groups, not only with one another but with water and dissolved ions.</font> </p>
        <p><font face="Arial, Helvetica, sans-serif">In a water-soluble polypeptide,
                most but not necessarily all of the surface R-groups will be
            hydrophilic. Hydrophobic
            groups will tend to be buried in the polypeptide's interior.&nbsp; </font></p>
        <p><font face="Arial, Helvetica, sans-serif">Some
            polypeptides are inserted into membranes. </font><font face="Arial, Helvetica, sans-serif">In
            these polypeptides, hydrophobic R-groups on the surface of the folded
            polypeptide will interact with the hydrophobic interior of the lipid
            bilayer.</font> (Does that make sense to you, thermodynamically?)</p>
        <hr align="center" width="60%" size="1" noshade />
<p><font face="Arial, Helvetica, sans-serif"><b>Other factors in protein
                    structure:&nbsp; </b>In addition to hydrophobic/hydrophilic
                    effects, ionic interactions between acidic and basic R-groups,
                    van der Waals interactions and H-bonds also play an important
                    role in determining the tertiary or three dimensional structure
            of the polypeptide. </font></p>
    </td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td>&nbsp;</td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="94"><p><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/vitamin-B12.jpg" width="96" height="120" alt="b12" /></p></td>
    <td width="13">&nbsp;</td>
    <td width="613"><p><font face="Arial, Helvetica, sans-serif">Proteins can
                include non-amino acid-based components, known generically as
                co-factors.&nbsp; A
                protein minus its cofactors is known as an apoprotein.&nbsp; Together
                with its cofactors, it is known as a holoprotein. Generally,
                without its cofactors, a protein is inactive (and often unstable). </font></p>
        <p><font face="Arial, Helvetica, sans-serif">Cofactors can range in complexity
                from a single metal atom to quite complex molecules, such as <a href="http://upload.wikimedia.org/wikipedia/commons/thumb/e/ec/Cobalamin.png/280px-Cobalamin.png" target="_blank">&larr;
                vitamin B<sub>12</sub></a></font>.&nbsp; The retinal group
        of bacteriorhodopsin and the heme group (with its central iron ion, are
        also a co-factors.&nbsp; </p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td>&nbsp;</td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td><p><font face="Arial, Helvetica, sans-serif">In general, a gene does
        not necessarily encode a protein, but rather a polypeptide.&nbsp; A
        protein is a functional entity. In many cases a protein is composed
        of multiple polypeptides. </font></p></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td>&nbsp;</td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td width="571" height="13"><p>For
                example, the membrane bound ATP synthase &rarr; we talked about
                previously is composed of&nbsp; 8
                different types of polypeptides, many of which are present in
                multiple copies assembled to form  the final, functional protein.&nbsp; </p>
        <p><font face="Arial, Helvetica, sans-serif">A protein that contains
            multiple polypeptides, whether of the same or different types, is
            said to have a quaternary</font> structure.        </p>
        <p align="left"><font face="Arial, Helvetica, sans-serif">There are also
            higher levels of polypeptide organization. For example, a particular
        polypeptide may be a component of a number of different proteins. </font></p></td>
    <td width="10">&nbsp;</td>
    <td width="119"><a href="http://www.atpsynthase.info/FAQ.html" target="_blank"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ATPsynthase.jpg" width="120" height="160" alt="atp synthase" /></a></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0" bgcolor="#CCCCCC">
  <tr>
    <td width="80" bgcolor="#990000"><div align="center">
      <p class="white">Questions to answer</p>
      <p class="white">&nbsp;</p>
      <p class="white"><br />
      </p>
    </div></td>
    <td width="640" bgcolor="#FFFFFF"><font face="Arial, Helvetica, sans-serif">&nbsp; </font>
      <ol>
        <li><span class="style5">How might   changing the pH of the solution
          alter a protein's structure and activity?</span></li>
        <li>What happens to a typical protein if you place it in a hydrophobic
          solvent? </li>
        <li>What would be the structure of a polypeptide if all of its R-groups
          were hydrophilic?&nbsp;</li>
        <li>How might a chaperone recognize a misfolded polypeptide?&nbsp; </li>
        <li>What
          might the role of chaperones be in the assembly of a protein composed
          of multiple polypeptides?</li>
        <li>Summarize the differences in structure between a protein that is
          soluble in the cytoplasm and one that is buried in the membrane. </li>
        <li>Why might proteins that require co-factors misfold in the absence
          of the co-factor? </li>
        <li>How could surface hydrophobic  R-groups facilitate protein-protein
          interactions.</li>
      </ol></td>
  </tr>
  <tr>
    <td bgcolor="#0000FF"><div align="center"><span class="whoe1"><span class="white">Questions to ponder</span></span><br />
    </div></td>
    <td bgcolor="#FFFFFF"><ul>
      <li class="style21">How does increasing entropy drive protein assembly? </li>
      <li class="style21">Suggest some reasons why cofactors are necessary in biological systems (proteins)? </li>
    </ul></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" /></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><iframe width="720" height="574" frameborder="0" src="http://besocratic.clemson.edu/besocratic.html?Activity=811#/ActivityViewerPage"></iframe></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
<tr>
      <td width="21"><a href="Topic3-4_Mutation.htm" target="_self"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="" width="24" height="24" border="0" /></a></td>
    <td width="55"><div align="left"><a href="#TOP"><font face="Arial, Helvetica, sans-serif" size="-2"> <img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowUP.GIF" width="24" height="24" /></font></a></div></td>
    <td width="615"><div align="right"><font size="-2" face="Arial, Helvetica, sans-serif">modified  
      <!-- #BeginDate format:En2 -->31-Aug-2012<!-- #EndDate -->
    </font></div></td>
    <td width="29"><div align="right"><a href="Topic3-6_Making Proteins.htm" target="_self"><img src="../../../Biofundamentals/BioFun-Support/AllGraphics/ArrowRIGHT.gif" width="24" height="24" border="0" /></a></div></td>
  </tr>
</table>

<script type='text/javascript'>var _hl=_hl||{};_hl.site='41';(function(){var hl=document.createElement('script');hl.type='text/javascript';hl.async=true;hl.src=document.location.protocol+'//highlighter.com/webscript/v1/js/highlighter.js';var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(hl,s);})();</script>


</body>>
</html>
