<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head><meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
   <meta name="Content-Type" content="text/html; charset=iso-8859-1" />
   <meta name="description" content="Interactive Introductory / Foundational Evolution and Molecular Biology" />
   <meta name="keywords" content="free on-line biology text, interactive molecular biology, interactive text, biology on-line" /><meta name="" content="ffree on-line biology, molecular biology, text" />
   <meta name="keywords" content="Biofundamentals, College-level, Foundational concepts, Klymkowsky, Evolution, Molecular Biology, Cell Biology, Interactive" />
   <meta name="GENERATOR" content="Mozilla/4.75 (Macintosh; U; PPC) [Netscape]" />
   <title>Regulatory Networks - Biofundamentals @ UC Boulder </title>


<script language="JavaScript" type="text/JavaScript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}

function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}

function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}

//-->
</script>
</head>
<style type="text/css">
<!--
a:link {text-decoration: none}
a:visited {text-decoration: none}
a:active {text-decoration: none}
a:hover {text-decoration: underline}
.style2 {font-family: Arial, Helvetica, sans-serif}
body,td,th {
	font-family: Arial, Helvetica, sans-serif;
}
.style3 {
	color: #990000;
	font-weight: bold;
}
.style4 {font-size: x-small}
.style5 {font-weight: bold}
.white {
	color: #FFF;
}
-->
</style>

<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="667"><font face="Arial, Helvetica, sans-serif"><b>26. Regulatory
        networks</b><font color="#006600"><b><a name="top"></a></b></font></font></td>
    <td width="28"><a href="Topic4-1_GeneExp.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowLEFT.GIF" alt="arrow" width="24" height="24" border="0" /></a></td>
    <td width="25"><div align="right"><a href="Topic5-1_CellDivision.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowRIGHT.gif" alt="arrow" width="24" height="24" border="0" /></a></div></td>
    <table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" noshade /></td>
  </tr>
</table>
  </tr>
</table>

<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif"> Inside the cell, DNA exists
          in association with various proteins.&nbsp; This DNA-protein complex
          is known as chromatin</font>. </p>
      <p><font face="Arial, Helvetica, sans-serif">Based on the structure of
              DNA, each base pair is about 0.34 nm in length. A kilobase
              (that is, 10<sup>3</sup> base
              pairs) of DNA is therefore about 0.34 &micro;m
            in length.&nbsp;&nbsp; A
            bacterium, like
          <em>E. coli</em>, has ~ 3 x 10<sup>6</sup> base pairs of
            DNA &ndash; that is a DNA molecule almost
        a millimeter in length.&nbsp;</font></p></td>
    <td>&nbsp;</td>
    <td><img src="../../BioFun-Support/wordles/page26.jpg" width="320" height="150" alt="page 19" /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td>&nbsp;</td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td><div align="left"><font face="Arial, Helvetica, sans-serif">The bacterial
                    cell </font><font face="Arial, Helvetica, sans-serif"> itself
                    is only about 0.8 &micro;m in diameter and about 2 &micro;m
                    in length. </font>&nbsp;<font face="Arial, Helvetica, sans-serif">A
                    human cell has about 6000 times more DNA, that is a total
                    length of greater than 2 meters.&nbsp; To  fit
            the DNA molecule into a cell takes some packing.&nbsp;The
                way the DNA is assembled into chromatin, particularly in
                eukaryotic cells, can dramatically influence the ability
                of transcription factors to interact with and bind to regulatory
        sequences.&nbsp;</font></div></td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td>&nbsp;</td>
    </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td width="242" height="407"><font face="Arial, Helvetica, sans-serif"><img src="../../BioFun-Support/AllGraphics/openingChromatin.jpg" width="241" height="400" /></font></td>
    <td width="10">&nbsp;</td>
    <td width="468"><p><font face="Arial, Helvetica, sans-serif">R</font><font face="Arial, Helvetica, sans-serif">egions
          of chromatin that are packaged so as to be   inaccessible
          to regulatory proteins are known as heterochromatin</font>. </p>
        <p><font face="Arial, Helvetica, sans-serif">Chromatin that is accessible
            and transcriptionally active is known as euchromatin</font>. </p>
        <p><font face="Arial, Helvetica, sans-serif">Some regions of chromatin
            are folded away more or less permanently; they  are unfolded only
            during DNA replication.&nbsp; These are known as constitutive
            heterochromatin</font>.&nbsp; <font face="Arial, Helvetica, sans-serif">Other
            regions can be unpacked in different cell types and are known as facultative
            heterochromatin</font>. </p>
        <p><font face="Arial, Helvetica, sans-serif">A particularly dramatic
            example of this process occurs in female mammals.&nbsp; One of the
            two X chromosomes is packed into a heterochromatic state, known as
            a <a href="http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/S/SexChromosomes.html#x-inactivation" target="_blank">Barr
            body</a>. </font></p>
        <p><font face="Arial, Helvetica, sans-serif">Most of the genes on this
        heterochromatic X chromosome are not expressed; so even though females
                (XX) have twice as many of the genes located on the X chromosome
                as males (XY), both make about the same amount of gene product. </font></p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif">Transcription factors also
          influence the proteins that act on chromatin to make the DNA within
          it accessible or inaccessible.&nbsp;&nbsp; This regulation of chromatin
          packing can act on blocks of DNA, regulating all of the genes within
          the affected region.&nbsp; </font></p>
    <p><font face="Arial, Helvetica, sans-serif">In the end, the gene regulatory
      elements within the DNA compete with one another for the binding of DNA-dependent,
            RNA polymerases.&nbsp; Genes that assemble the most efficient promoters
            are the most effective at loading RNA polymerase molecules. </font><font face="Arial, Helvetica, sans-serif"> These
            genes will produce more copies of the RNA that they encode.</font></p>
    </td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p align="left"><font face="Arial, Helvetica, sans-serif"><strong>Networks</strong>:&nbsp; A
                cell's behavior will be determined in large part by which of
                its genes are actively expressed.&nbsp; </font></p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">How can a cell
          determine that the &quot;right&quot; set of genes is activated for
          a particular situation? </font> Clearly feed back systems are needed. </p>
      <p align="left"><font face="Arial, Helvetica, sans-serif">Genes that need
          to be expressed together to perform a function can be co-regulated
          by using similar sets of regulatory sequences. </font></p></td>
    <td>&nbsp;</td>
    <td><img src="../../BioFun-Support/AllGraphics/GeneticCircuit.jpg" width="260" height="168" /></td>
  </tr>
</table>
<table width="700" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><font face="Arial, Helvetica, sans-serif"><a href="http://jura.wi.mit.edu/cgi-bin/young_public/navframe.cgi?s=17&f=" target="_blank">A
          comprehensive analysis</a> of  the
        interactions between 10<sup>6</sup> transcription factors  and
        regulatory sequences  in
        the baker's yeast <em>Saccharomyces cerevisiae</em>  revealed the presence
    of a number of regulatory motifs. </font></td>
  </tr>
</table>
<table width="700" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td width="89"><div align="center"><a href="http://jura.wi.mit.edu/cgi-bin/young_public/navframe.cgi?s=17&f=autoreg" target="_blank"><img src="../../BioFun-Support/AllGraphics/autoreg.gif" width="62" height="100" alt="autoregulatory interaction" /></a></div></td>
    <td width="482"><font face="Arial, Helvetica, sans-serif"><strong>Autoregulatory
    loops: </strong> In this type of regulatory motif, a transcription
    factor binds to regulatory sequences that regulate its own transcription.&nbsp; Such
    interactions can be positive (amplifying) or negative (squelching).&nbsp; </font></td>
    <td width="89"><div align="center"><img src="../../BioFun-Support/AllGraphics/autoregNeg.gif" width="62" height="100" /></div></td>
  </tr>
  <tr>
    <td>&nbsp;</td>
    <td>&nbsp;</td>
    <td>&nbsp;</td>
  </tr>
  <tr>
    <td><div align="center"><a href="http://jura.wi.mit.edu/cgi-bin/young_public/navframe.cgi?s=17&f=feedforward" target="_blank"><img src="../../BioFun-Support/AllGraphics/feedforward.gif" width="89" height="124" border="0" /></a></div></td>
    <td><font face="Arial, Helvetica, sans-serif"><strong>Feed forward interactions:  </strong> A
        transcription factor regulates the expression of a second transcription
        factor; the two transcription factors then cooperate to regulate the
        expression of a third gene.&nbsp; </font></td>
    <td><div align="center"><img src="../../BioFun-Support/AllGraphics/feedforward%20negative.gif" width="89" height="124" /></div></td>
  </tr>
  <tr>
    <td>&nbsp;</td>
    <td>&nbsp;</td>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td width="594"><table width="100%"  border="0" cellspacing="0" cellpadding="0">
      <tr>
        <td><a href="http://jura.wi.mit.edu/cgi-bin/young_public/navframe.cgi?s=17&f=multicomp" target="_blank"><img src="../../BioFun-Support/AllGraphics/multicomp.gif" width="87" height="86" border="0" /></a></td>
        <td><font face="Arial, Helvetica, sans-serif"><strong>&larr; Multicomponent
              loops: </strong> Transcription factors act on regulatory sequences
              in a close loop arrangement.&nbsp;&nbsp; </font></td>
      </tr>
    </table>      
      <p><font face="Arial, Helvetica, sans-serif"><strong>Regulatory chains:&rarr; </strong> In
            this arrangement, a transcription factor binds to regulatory sequences
            in another gene and induces expression of a second transcription
          factor, which in turn binds to regulatory sequences in a third gene,
      etc.&nbsp; </font></p>
      <p><font face="Arial, Helvetica, sans-serif">The
              chain ends with the production of some non-transcription factor
    products (if not, it is probably a multicomponent loop).&nbsp; </font></p></td>
    <td width="66"><div align="center"><a href="http://jura.wi.mit.edu/cgi-bin/young_public/navframe.cgi?s=17&f=cascade" target="_blank"><img src="../../BioFun-Support/AllGraphics/regulator%20chain.gif" width="47" height="267" border="0" /></a></div></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><div align="center">
        <p><a href="http://jura.wi.mit.edu/cgi-bin/young_public/navframe.cgi?s=17&f=mim" target="_blank" onMouseOver="MM_swapImage('Image19','','../../BioFun-Support/AllGraphics/multiinput%20module.gif',1)" onMouseOut="MM_swapImgRestore()"><img src="../../BioFun-Support/AllGraphics/single input module.gif" width="141" height="97" alt="single input" /><br />
            <img src="../../BioFun-Support/AllGraphics/multiinput module.gif" width="141" height="97" alt="multi-input" /></a></p>
    </div></td>
    <td><p><font face="Arial, Helvetica, sans-serif"><strong>&larr; Single and
                    multiple input modules: </strong> A transcription factor
                    binds to sequences in a number of genes, regulating their
                    coordinated expression.&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif">In
            most cases, sets of target genes are regulated by sets of transcription
            factors that bind in concert.&nbsp;</font></p>
<p><font face="Arial, Helvetica, sans-serif"><strong>Regulation of regulatory
                    networks:&rarr;&nbsp;</strong> Transcription factors are
                    proteins, and so their stability and activities can be regulated
                    by interactions with other proteins, allosteric factors,
                    and post-translational modifications. &nbsp;</font><font face="Arial, Helvetica, sans-serif">It
                    is through such interactions that signals from outside the
    cell can alter the patterns of gene expression. </font></p></td>
    <td><a href="http://jura.wi.mit.edu/cgi-bin/young_public/navframe.cgi?s=17&f=autoreg" target="_blank"><img src="../../BioFun-Support/AllGraphics/autoregNeg.gif" width="62" height="100" alt="auto-on" /></a><br />        <img src="../../BioFun-Support/AllGraphics/autoregNeg-RegOff.gif" width="62" height="100" alt="auto-off" /></td>
  </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
    <tr>
    <td>&nbsp;</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><p><font face="Arial, Helvetica, sans-serif">At the same time, internal
          signals, generated by metabolic processes, can also feedback to the
          gene regulatory system in order to maintain the homeostatic state.&nbsp; </font></p>
        <p><font face="Arial, Helvetica, sans-serif">In fact, such feedback is
            critical to maintaining  regulatory networks &ndash; inhibiting
        them from spiraling out of control &ndash; which would lead to death. </font>&nbsp;</p></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" noshade /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
        <td><p><font face="Arial, Helvetica, sans-serif"><strong>Genome and gene
                        duplication</strong>: </font></p>
                <p><font face="Arial, Helvetica, sans-serif">An obvious question
                        is, how can biological systems increase their complexity?
                        &nbsp;Where do new genes and new activities come from?&nbsp; </font></p>
                <p><font face="Arial, Helvetica, sans-serif">One mechanism is
                        by duplicating pre-existing genes [<a href="http://biology.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pbio.0020206" target="_blank">link</a>].&nbsp;&nbsp; </font><font face="Arial, Helvetica, sans-serif">Typically
                        selection will act to maintain the function of one copy
                        of the gene (which was  important in the organism
                        before the duplication and remains so afterward).&nbsp; The
                        second copy is then freer to change.&nbsp; Mutations
                        may lead to the loss of a functional gene product, or
                        a gene product with altered properties.&nbsp;</font></p>
        </td>
        <td><a href="http://biology.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pbio.0030344" target="_blank"><img src="../../BioFun-Support/AllGraphics/vertebrate genome duplication.jpg" width="240" height="220" /></a></td>
    </tr>
</table>
<table width="700" border="0" cellpadding="0" cellspacing="0">
    <tr>
        <td>&nbsp;</td>
    </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
    <tr>
        <td><p>There a number of different types of duplication events, which
                range from the duplication of a small region of a specific chromosome
                to the duplication of the entire genome (we will not go into
                the molecular details involved).&nbsp;&nbsp; The
                advent of genomic sequencing and the comparison of genomes from
                different organisms suggests that many types of organisms have
                undergone both processes.&nbsp; </p>
            <p>The most dramatic type of duplication event is the duplication
                of an entire genome.&nbsp; After
                such an event (which is rare), some genes will be lost.&nbsp; This
                will be the result of random mutations that inactivate the gene.&nbsp; Since
                there is another copy of the gene, the loss of the second copy
                may have little effect. </p>
            <p>On the other hand, when a gene is duplicated, it is likely that
                the total amount of gene product produced will increase - in
                some cases, increased gene expression can have harmful effects
                (think mammalian X chromosome and females).
                When this is the case, inactivation of one copy of the
               gene may, in fact, be beneficial.&nbsp;</p>
            <p>Mutations can also lead to changes in activity and specificity
                - a mutation can change a protein so that it can bind new substrates
                and catalyze new reactions, a process known as neofunctionalization.&nbsp;&nbsp; </p>
            <p>One issue with &quot;blind&quot; duplication events is that the
                development of new functions occurs only after duplication.&nbsp; The
                duplicated gene may be lost through mutation (or genetic drift).
               &nbsp; <br />
               <br />
               An alternative process involves genes that develop potentially
                valuable &quot;side&quot;
                activities. Before a duplication event, the evolution
                of these side functions is constrained by the fact that the gene
                already has important functions to carry out.&nbsp; If the gene
                is duplicated, however, then the new function can be modified
                through the standard process of mutation and selection, while
                selection (conservative) acts to maintain the original function.
            &nbsp;</p>
<p>As we look at the genes of various organisms, we find evidence
                for both gene and genome duplication.&nbsp; For example, compared
                to their invertebrate ancestors, the vertebrate lineage appears
      to have gone through two rounds of whole genome duplication.&nbsp; </p></td>
    </tr>
</table>
<table width="700" border="0" cellspacing="0" cellpadding="0">
    <tr>
    <td><hr size="1" noshade /></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0" bgcolor="#CCCCCC">
  <tr>
    <td width="78" bgcolor="#990000"><div align="center">
      <p class="white">Questions to answer      </p>
      <p class="white">&nbsp;</p>
      <p class="white">&nbsp;</p>
      <p class="white"><br />
      </p>
    </div></td>
    <td width="622" bgcolor="#FFFFFF"><ol>
        <li>How could you tell whether the same X chromosome was
          inactivated in each cell of a female person? </li>
        <li>How would you design a multicomponent loop to produce
          a steady level of product?</li>
        <li>How would  the presence of a repressor factor influence
          the various modules described above?</li>
        <li>How can transcription factors be regulated?</li>
        <li>How does regulating the intracellular localization
          of a transcription factor alter gene expression? &nbsp;</li>
        <li><span class="style21">What kind of mutation would permanently inactivate
          a gene? </span></li>
      </ol></td>
  </tr>
  <tr>
    <td bgcolor="#0000FF"><div align="center"><span class="whoe1"><span class="white">Questions to ponder</span></span><br />
    </div></td>
    <td bgcolor="#FFFFFF"><ul>
      <li class="style21"><span class="style2">Why,
          do you think, that one of the two female X chromosomes is packed
          into heterochromatin?&nbsp; What
        is the point of that? &nbsp; </span></li>
      <li class="style21"><span class="style2">What type of event would lead to total genome
        duplication? </span></li>
      <li class="style21"><span class="style2">Why are some genes lost after genome duplication? </span></li>
    </ul></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td><hr size="1" noshade /></td>
  </tr>
</table>
<table width="720" border="0" cellspacing="0" cellpadding="0">
  <tr>
    <td>replace with revised beSocratic activity</td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td><hr size="1" /></td>
  </tr>
</table>

<table width="720" border="0" cellpadding="0" cellspacing="0">
  <tr>
    <td width="33"><div align="left"><a href="#TOP"><font face="Arial, Helvetica, sans-serif" size="-2"> <img src="../../BioFun-Support/AllGraphics/ArrowUP.GIF" width="24" height="24" /></font></a></div></td>
    <td width="534"><div align="right"><font size="-2" face="Arial, Helvetica, sans-serif"> <a href="https://twitter.com/share" class="twitter-share-button">Tweet</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>&nbsp;&nbsp; revised
          <!-- #BeginDate format:En2 -->10-May-2014<!-- #EndDate -->
    </font></div></td>
    <td width="25"><div align="right"><a href="Topic5-1_CellDivision.htm" target="_self"><img src="../../BioFun-Support/AllGraphics/ArrowRIGHT.gif" width="24" height="24" border="0" /></a></div></td>
  </tr>
</table>

<script type='text/javascript'>var _hl=_hl||{};_hl.site='2622';(function(){var hl=document.createElement('script');hl.type='text/javascript';hl.async=true;hl.src=document.location.protocol+'//highlighter.com/webscript/v1/js/highlighter.js';var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(hl,s);})();</script>

</body>
</html>